<html>
<!-- ####################################################### -->
<!-- #  This html was produced by the igemwiki generator   # -->
<!-- #  https://github.com/igemuoftATG/generator-igemwiki  # -->
<!-- ####################################################### -->

<!-- repo for this wiki: https://github.com/igemuoftATG/wiki2015 -->

</html>
{{Toronto/head}}
<html>

<div id="navigation">
<ul>
<li><a href="http://2015.igem.org/Team:Toronto"><span>home</span><i class="fa fa-home"></i></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Team"><span>team</span><i class="fa fa-users"></i></a></li>
</li>
<li class="active"><a href="#"><span>project</span><i class="fa fa-info-circle"></i></a>
<ul>
<li><a href="http://2015.igem.org/Team:Toronto/Description"><span>description</span></a></li>
</li>
<li class="active"><a href="http://2015.igem.org/Team:Toronto/Experiments"><span>experiments</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Results"><span>results</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Design"><span>design</span></a></li>
</li>
</ul>
<li><a href="#"><span>parts</span><i class="fa fa-gear"></i></a>
<ul>
<li><a href="http://2015.igem.org/Team:Toronto/Parts"><span>parts</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Basic_Part"><span>basic parts</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Composite_Part"><span>composite parts</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Part_Collection"><span>part collection</span></a></li>
</li>
</ul>
<li><a href="http://2015.igem.org/Team:Toronto/Notebook"><span>notebook</span><i class="fa fa-book"></i></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Attributions"><span>attributions</span><i class="fa fa-life-saver"></i></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Collaborations"><span>collaborations</span><i class="fa fa-users"></i></a></li>
</li>
<li><a href="#"><span>practices</span><i class="fa fa-tree"></i></a>
<ul>
<li><a href="http://2015.igem.org/Team:Toronto/Practices"><span>human practices</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Alberta-Oil-Stories"><span>alberta oil stories</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Community-Outreach"><span>community outreach</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Technology-Applications"><span>technology applications</span></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/TechnoMoral-Scenarios"><span>technoMoral scenarios</span></a></li>
</li>
</ul>
<li><a href="http://2015.igem.org/Team:Toronto/Safety"><span>safety</span><i class="fa fa-fire"></i></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Modeling"><span>modeling</span><i class="fa fa-cubes"></i></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Measurement"><span>measurement</span><i class="fa fa-bar-chart"></i></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Software"><span>software</span><i class="fa fa-code"></i></a></li>
</li>
<li><a href="http://2015.igem.org/Team:Toronto/Entrepreneurship"><span>entrepreneurship</span><i class="fa fa-institution"></i></a></li>
</li>
</ul>
</div>
<div class="content" id="content-main"><div class="row"><div class="col col-lg-8 col-md-12"><div class="content-main"><h1 id="experiments">Experiments</h1>
<h2 id="protocol-for-lb-medium-and-lb-agar-plates">Protocol for LB medium and LB agar plates</h2>
<p><em>Written by Katariina Jaenes</em></p>
<h3 id="lb-introduction">LB Introduction</h3>
<p>In order for bacteria to be successfully cultured, they must be grown in the appropriate media. LB, also
known as Lysogeny broth, is a nutrient rich broth that is a standard for culturing Escherichia coli, as it
allows for quick growth and high yields. Therefore the proper preparation of LB will be crucial to
maintaining our bacterial stock throughout the summer. Furthermore, addition of agar to LB broth
creates a gel for bacteria to grow upon, and is therefore used for plating bacterial cultures on petri
dishes.
We will be preparing our own LB as well as our own LB agar plates, which will be used to culture
E.coli K-12.</p>
<h3 id="lb-basic-terminology-and-concepts">LB Basic Terminology and Concepts</h3>
<p>Agar vs. Agarose: Agar is used for making petri plates to culture organisms, while agarose is used for
making gels, in the likes of SDS-PAGE and gel electrophoresis</p>
<h3 id="lb-safety-precautions">LB Safety Precautions</h3>
<p><strong>SDS (safety data sheet)</strong>: Refer to the SDS sheets for all listed materials before entering the lab. Be
prepared to answer any questions regarding the information on these sheets.</p>
<p><strong>PPE (Personal protective equipment)</strong>: Proper lab attire should be worn throughout the experiment: This
means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact
lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times,
and long hair should be tied back.</p>
<p><strong>Autoclave</strong>: The autoclave should only be handled by execs. <strong>Note that any autoclaved materials
may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</strong></p>
<h3 id="lb-materials-and-equipment">LB Materials and Equipment</h3>
<p>Note: this recipe is for 500ml of LB, which makes approximately 18 plates. LB can be kept for 3-4
months</p>
<h3 id="lb-reagents">LB Reagents</h3>
<ul>
<li>5g NaCl</li>
<li>5g Tryptone</li>
<li>2.5g Yeast extract</li>
<li>7.5g of agar <strong>This is only necessary if you are making LB agar plates</strong></li>
<li>500ml of dH20 (distilled water)</li>
</ul>
<h3 id="lb-materials">LB Materials</h3>
<ul>
<li>1L pyrex bottle</li>
<li>1L graduated cylinder</li>
<li>Filter paper and scupula</li>
<li>Stack of sterile plates (this protocol makes approx 25.)</li>
<li>Bunsen Burner/Ethanol burner</li>
<li>70% EtOH wash bottle</li>
<li>paper towels/wipes</li>
</ul>
<h3 id="lb-lab-protocol">LB Lab Protocol</h3>
<h4 id="-part-1-making-the-lb-broth-"><strong>Part 1. Making the LB broth</strong></h4>
<p><em>This step can be carried out at a regular lab bench.</em></p>
<ol>
<li>Obtain a clean 1L pyrex bottle</li>
<li>Obtain a graduated cylinder with 500ml of dH20 (distilled water) and add to the bottle. Record the
amount added.</li>
<li>Using filter paper, separately measure out 5g of NaCl, 5 g of Tryptone, and 2.5g of yeast extract on a
scale and add them to the bottle. Swirl the bottle in a circular motion to mix. Remember to recalibrate
your scales in between measurements.</li>
<li>If you are making LB agar plates, weigh and add 7.5g of agar and swirl to mix. Record the
amount added. Note that the contents do not necessarily need to be completely in solution before
autoclaving.</li>
</ol>
<h4 id="-part-2-autoclaving-only-done-by-execs-"><strong>Part 2. Autoclaving *</strong>ONLY DONE BY EXECS<strong>*</strong></h4>
<ol>
<li>Lightly seal the top of the beaker with aluminium foil, and label the beaker with autoclave tape.
Include LB (agar) – your name – date - iGEM.
<strong>Unless you have been trained to use the autoclave, you will not be conducting the following
steps 2 - 5.</strong></li>
<li>Use appropriate transportation protocols to bring the LB bottle into the autoclave room. Remember
to store the beaker in an autoclavable basin, in case of spills.</li>
<li>Check the water level on the autoclave, if necessary. Autoclave on the liquid setting for
approximately 20mins.</li>
<li><strong>The contents of the beaker will be hot after autoclaving, therefore take necessary measures
to prevent burns.</strong> After autoclaving, allow the LB media too cool to 55oC before handling.</li>
<li>The LB broth can be stored in sterile conditions at room temperature, and should be good for a 3-4
months. Flame the lip of the bottle each time the LB is used. If the LB is contaminated it will appear cloudy. If the LB contains antibiotics: store in a
-4C freezer - (However it is not recommended to store LB with antibiotics as the antibiotic will degrade over time)</li>
</ol>
<h4 id="-part-3-pouring-the-plates-if-you-are-making-lb-agar-"><strong>Part 3. Pouring the plates (if you are making LB agar)</strong></h4>
<p><strong> While pouring the plates it is crucial to maintain a sterile environment. This should be done in
room WB303, with a sterile environment provided by a lit bunsen burner.</strong></p>
<p><em>Note: While you are waiting for the autoclave, steps 1- 3 can be done in the meantime, in addition to
the clean up from Part 1.</em></p>
<ol>
<li>Sterilize the workspace with 70% EtOH before depositing your materials. Light the bunsen burner.</li>
<li>Obtain a stack/roll of empty plates. The plates should still be in their plastic sleeve/wrapping, as they
should be sterile. Don&#39;t throw out the wrapping as it can be used to store the plates. It is essential that
you minimize any chance of contaminating the plates. Make sure that you open the package at the
top, and expose the plates as minimally as possible. Note that this protocol makes approximately 25
plates.</li>
<li>Once you take the plates out, store them upside down on your lab bench. Label the plates with your
name – iGEM 2015 – date prepared – designated stripe* (if are using antibiotic). Once labelled, you
may stack the plates to free up workspace.</li>
<li>Allow the LB media to cool before pouring. The LB will start to settle at about 30oC.</li>
<li>If you are preparing selective media, add antibiotic to the mixture. Use the recommended antibiotic
concentrations (see table below) Swirl the flask in a circular motion to mix. If you don&#39;t know
whether or not you are preparing selective media, ask.</li>
</ol>
<table>
<thead>
<tr>
<th style="text-align:center">Antibiotic</th>
<th style="text-align:center">Recommended Concentration</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center">Chloramphenicol (CAM)</td>
<td style="text-align:center">25ug/mL</td>
</tr>
<tr>
<td style="text-align:center">Ampicillin (AMP)</td>
<td style="text-align:center">100ug/mL</td>
</tr>
</tbody>
</table>
<p><em>One stripe along the sides corresponds to CAM, two stripes corresponds to AMP</em></p>
<ol>
<li>Take an empty plate and open it slightly. You do not need to open it all the way to pour the agar.
When pouring the agar, pour until 2/3 of the plate has been covered, or approximately half of the
plate has been filled when viewed from the side. Pour the agar slowly, to prevent the formation of
bubbles. Swirl the plate in a circular motion to distribute the media evenly on the plate.</li>
<li>After pouring, set the plates to cool in stacks of 4-5, as this takes up less space. Don&#39;t stack the plates
too high – we want to prevent spills.</li>
<li>Rinse the pyrex bottle with water before the remnants solidify and become hard to remove.</li>
<li>Allow the plates to cool for at least 20 minutes until the agar has solidified. Flip the plates to
prevent condensation forming on the agar. The plates can then be stacked and stored in plastic bags
(ideally re-use the plastic bags that the plates come in.)</li>
<li>The LB agar plates should be stored in a 4oC freezer, should be good for 1-2 months</li>
</ol>
<h3 id="lb-leaving-the-lab">LB Leaving the lab</h3>
<p>Prior to leaving the lab, you should:</p>
<ol>
<li>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</li>
<li>Wipe down your workspace.</li>
<li>Ensure that all materials have been returned to their places, and that the plates have been properly
stored in the fridge.</li>
</ol>
<h3 id="lb-references">LB References</h3>
<p>Addgene: Protocol – Making LB Agar Plates for Bacteria. From: <a href="https://www.addgene.org/plasmid-
protocols/bacterial-plates/">https://www.addgene.org/plasmid-
protocols/bacterial-plates/</a></p>
<p>Department of Ecology and Evolutionary Biology, UCLA. Making LB Agar Plates. From
<a href="https://www.eeb.ucla.edu/Faculty/Barber/Web%20Protocols/LB%20Agar%20Plates.pdf">https://www.eeb.ucla.edu/Faculty/Barber/Web%20Protocols/LB%20Agar%20Plates.pdf</a></p>
<p>U of T iGEM Luria Broth (LB) protocol</p>
<p>Sezonov G, Joseleau-Petit D, D’Ari R. Escherichia coli Physiology in Luria-Bertani Broth . Journal of
Bacteriology. 2007;189(23):8746-8749. doi:10.1128/JB.01368-07.</p>
<h2 id="protocol-for-soc-medium">Protocol for SOC medium</h2>
<p><em>Written by Katariina Jaenes</em></p>
<h3 id="soc-introduction">SOC Introduction</h3>
<p>SOC is a variant of the rich media SOB (super optimal broth) with catabolite repression. This means
that glucose is supplemented in the media, allowing for optimal metabolic conditions for the bacteria.
SOC increases the transformation efficiency of cells, as it provides ample nutrients to cells that have
recently undergone stress as result of having been made competent. Accordingly, it will be used in
bacterial transformation to stabilize the cells and to increase transformation yields. Since SOC is high
in nutrients, it is more easily contaminated than LB or King&#39;s B media.</p>
<h3 id="soc-safety-precautions">SOC Safety Precautions</h3>
<p><strong>SDS (safety data sheet)</strong>: Refer to the SDS sheets for all listed materials before entering the lab. Be
prepared to answer any questions regarding the information on these sheets.</p>
<p><strong>PPE (Personal protective equipment)</strong>: Proper lab attire should be worn throughout the experiment: This
means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact
lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times,
and long hair should be tied back.</p>
<p><strong>Autoclave</strong>: The autoclave should only be handled by execs. <strong>Note that any autoclaved materials
may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</strong></p>
<h3 id="soc-materials-and-equipment">SOC Materials and Equipment</h3>
<p><em>Note: this recipe is for 500mL of SOC.</em></p>
<p><strong>Reagents</strong></p>
<ul>
<li>1.802g glucose</li>
<li>10g tryptone</li>
<li>2.5g Yeast extract</li>
<li>0.584 g NaCl</li>
<li>0.093g KCl</li>
<li>1.016g MgCl2 (anhydrous)</li>
<li>1.234g MgSO4 .7H20</li>
<li>500ml of dH20 (distilled water)</li>
</ul>
<p><strong>Materials</strong></p>
<ul>
<li>2 x 1L pyrex bottle (must have cap)</li>
<li>Note: A smaller pyrex bottle may be used to accommodate the glucose solution.</li>
<li>1L graduated cylinder</li>
<li>weighing boats and scupula</li>
<li>70% EtOH wash bottle</li>
<li>paper towels/wipes</li>
</ul>
<h3 id="soc-lab-protocol">SOC Lab Protocol</h3>
<h4 id="-part-1-making-the-soc-broth-"><strong>Part 1. Making the SOC broth</strong></h4>
<p><em>This step can be carried out at a regular lab bench.</em></p>
<ol>
<li>Obtain two 1L mL pyrex bottles. Ensure that the bottles can be sealed with a cap - this will help
prevent contamination and enable long term storage.</li>
<li>Obtain a graduated cylinder with 500ml of dH20 (distilled water).</li>
<li>Using filter paper, separately measure out 10g tryptone, 2.5g yeast extract, 0.292g NaCl, 0.093g
KCl, 1.016g MgCl2 anhydrate, 1.234g MgSO4, heptahydrate. on a scale and add them to the 1L bottle.
Add 400mL dH20. Swirl the flask in a circular motion to mix. Remember to recalibrate your scales in
between measurements.</li>
<li>In the separate bottle, and measure out 1.802g glucose on a scale and add the rest of the 100mL
dH20. Recalibrate your scales in between measurements.</li>
</ol>
<p><strong><em>* The 1L autoclaved separately, as the contents will react if autoclaved together. </em></strong></p>
<h4 id="-part-2-autoclaving-only-done-by-execs-"><strong> Part 2.  Autoclaving ONLY DONE BY EXECS</strong></h4>
<ol>
<li>Lightly seal the top of the bottles with aluminium foil, or unscrew the caps. Label both with
autoclave tape. Include a label with SOC – your name – date - iGEM 2015.
<strong>Unless you have been trained to use the autoclave, you will not be conducting the following
steps 2 - 5.</strong></li>
<li>Use appropriate transportation protocols to bring the bottles into the autoclave room. Remember to
store the beaker in an autoclavable basin, in case of spills.</li>
<li>Check the water level on the autoclave, if necessary. Autoclave on the liquid setting for
approximately 20mins.</li>
<li><strong>The contents of the bottles will be hot after autoclaving, therefore take necessary measures
to prevent burns. After autoclaving, allow the media too cool to 55oC before handling.</strong></li>
</ol>
<h4 id="-part-3-"><strong>Part 3</strong></h4>
<p><em>This part should be done in room WB303, in sterile conditions close to a bunsen burner.</em></p>
<ol>
<li>In a sterile environment, slowly add the autoclaved 1.802g glucose and dH20 to the beaker
containing the autoclaved 1.016g MgCl2 and 1.234g MgSO4, 10g tryptone, 2.5g yeast extract, 0.292g
NaCl, 0.093g Kcl and dH20. Flame the lip of the bottle before transferring the contents. Swirl to mix,
and seal tightly to prevent contamination. Flame the cap before sealing.</li>
<li>The SOC broth can be stored in sterile conditions at room temperature, and should be good for a 3-4
months. Flame the lip of the bottle each time the SOC is used. SOC should be handled carefully, as it is
especially prone to contamination.</li>
</ol>
<h3 id="soc-leaving-the-lab">SOC Leaving the Lab</h3>
<p>Prior to leaving the lab, you should:</p>
<ol>
<li>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</li>
<li>Wipe down your workspace.</li>
<li>Ensure that all materials have been returned to their places, and that the plates have been properly
stored in the fridge.</li>
</ol>
<h3 id="soc-references">SOC References</h3>
<p>Cold Spring Harbour Protocols. SOC Medium for E.coli. From:
<a href="http://cshprotocols.cshlp.org/content/2012/6/pdb.rec069732.short">http://cshprotocols.cshlp.org/content/2012/6/pdb.rec069732.short</a></p>
<p>Sun QY et al. Culture of Escherichia coli in SOC medium improves the cloning efficiency of toxic
protein genes. Anal Biochem. 2009: From <a href="http://www.ncbi.nlm.nih.gov/pubmed/19622338">http://www.ncbi.nlm.nih.gov/pubmed/19622338</a></p>
<p>Qiagen. Recipe for SOC Medium. From: <a href="https://www.qiagen.com/ca/resources/faq?id=d3be05d0-
ec02-4ced-aabe-dcd2a1061920&amp;lang=en">https://www.qiagen.com/ca/resources/faq?id=d3be05d0-
ec02-4ced-aabe-dcd2a1061920&amp;lang=en</a></p>
<h2 id="king-s-b-protocol">King&#39;s B Protocol</h2>
<p><em>Written by Katariina Jaenes</em></p>
<h3 id="kb-introduction">KB Introduction</h3>
<p>In order for bacteria to be successfully cultured, they must be grown in the appropriate media. King&#39;s B
media, formulated by King et al,. Is a nonselective media that is a standard for culturing Pseudomonas
species, as it allows for quick growth and high yields. Therefore, the proper preparation of King&#39;s B
will be crucial to maintaining our bacterial stock throughout the summer. Furthermore, addition of agar
to King&#39;s B creates a gel for bacteria to grow upon, and is therefore used for plating bacterial cultures
on petri dishes.
We will be preparing our own King&#39;s B as well as our own King&#39;s B agar plates, which will be used to
culture Pseudomonas putida F1.</p>
<h3 id="kb-basic-terminology-and-concepts">KB  Basic Terminology and Concepts</h3>
<p>Agar vs. Agarose: Agar is used for making petri plates to culture organisms, while agarose is used for
making gels, in the likes of SDS-PAGE and gel electrophoresis</p>
<h3 id="kb-safety-precautions">KB Safety Precautions</h3>
<p><strong>SDS (safety data sheet)</strong>: Refer to the SDS sheets for all listed materials before entering the lab. Be
prepared to answer any questions regarding the information on these sheets.</p>
<p><strong>PPE (Personal protective equipment)</strong>: Proper lab attire should be worn throughout the experiment: This
means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact
lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times,
and long hair should be tied back.</p>
<p><strong>Autoclave</strong>: The autoclave should only be handled by execs. <strong>Note that any autoclaved materials
may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</strong></p>
<h3 id="kb-materials-and-equipment">KB Materials and Equipment</h3>
<p><em>Note: this recipe is for 500ml of LB, which makes approximately 18 plates. King&#39;s B can be kept for
3-4 months</em></p>
<p><strong>Reagents</strong></p>
<ul>
<li>5g proteose peptone</li>
<li>0.75g anhydrous K2HPO4</li>
<li>7.5g glycerol</li>
<li>1.6mL MgSO4 (of 1M solution )</li>
<li>500mL of dH20 (distilled water)</li>
<li>7.5g of agar <strong>This is only necessary if you are making King&#39;s B agar plates</strong></li>
</ul>
<p>Materials</p>
<ul>
<li>2 x1L pyrex bottle
◦ Note: the MgSO4 solution may kept in a smaller bottle</li>
<li>70% EtOH wash bottle</li>
<li>Filter paper and scupula</li>
<li>Stack of sterile plates (this protocol makes approx 25.)</li>
<li>paper towels/wipes</li>
<li>Bunsen burner/Ethanol burner</li>
</ul>
<h3 id="kb-lab-protocol">KB Lab Protocol</h3>
<p><em>makes 500mL of King&#39;s B, should be good for 3-4 months</em></p>
<h4 id="-part-1-making-king-s-b-broth-"><strong>Part 1. Making King&#39;s B broth</strong></h4>
<p>This step can be carried out at a regular lab bench.</p>
<ol>
<li>Obtain 2 clean 1L Erlenmeyer flask.</li>
<li>Obtain a graduated cylinder with around 500ml of dH20 (distilled water) and add it to one bottle.
Record the amount added.</li>
<li>Using filter paper, separately measure out 5g of proteose peptone, 0.75 g anhydrous K2HPO4
7.5 g glycerol on a scale and add them to the bottle with 500mL dH20. Swirl the beaker in a circular
motion to mix. Remember to recalibrate your scales in between measurements.</li>
<li>In a separate flask, add 12g of MgSO4
and 100mL of dH20. This will be the stock solution of 1M</li>
<li>If you are making agar plates, weigh and add 7.5g of agar to the 500mL solution, and swirl to mix.
Record the amount added. Note that the contents do not necessarily need to be completely in solution
before autoclaving.</li>
</ol>
<h4 id="-part-2-autoclaving-only-done-by-execs-"><strong>Part 2. Autoclaving *</strong>ONLY DONE BY EXECS<em>*</em></h4>
<ol>
<li>Lightly seal the top of the flask with aluminium foil, and label it with autoclave tape. Include King&#39;s
B (agar) – your name – date - iGEM 2015.
<strong>Unless you have been trained to use the autoclave, you will not be conducting the following
steps 2 - 5.</strong></li>
<li>Use appropriate transportation protocols to bring the flask into the autoclave room. Remember to
store the beaker in an autoclavable basin, in case of spills.</li>
<li>Check the water level on the autoclave, if necessary. Autoclave on the liquid setting for
approximately 20mins.</li>
<li><strong>The contents of the beaker will be hot after autoclaving, therefore take necessary measures
to prevent burns.</strong> After autoclaving, allow the media too cool to 55oC</li>
<li>After autoclaving, add 1.6 mL of sterile 1 M MgSO4 in sterile conditions. If it is added before
autoclaving, the media will appear cloudy.</li>
<li>The broth can be stored in sterile conditions at room temperature, and should be good for a 3-4
months. Flame the lip of the bottle each time the media is used.</li>
</ol>
<h4 id="-part-3-pouring-the-plates-if-you-are-making-agar-plates-"><strong>Part 3. Pouring the plates (if you are making agar plates)</strong></h4>
<p><em>While pouring the plates it is crucial to maintain a sterile environment. This should be done in WB303, in sterile conditions provided by a lit bunsen burner. Alternatively, can be done in 403 with an Ethanol burner</em></p>
<p>Note: While you are waiting for the autoclave, steps 1- 3 can be done in the meantime, in addition to
the clean up from Part 1.</p>
<ol>
<li>Sterilize the workspace with 70% EtOH before depositing your materials.</li>
<li>Obtain a stack/roll of empty plates. The plates should still be in their plastic sleeve/wrapping, as they
should be sterile. Don&#39;t throw out the wrapping as it can be used to store the plates. It is essential that
you minimize any chance of contaminating the plates. Make sure that you open the package at the
top, and expose the plates as minimally as possible. Note that this protocol makes approximately 18
plates.</li>
<li>Once you take the plates out, store them on the lab bench and keep them upside down. Label the
plates with King&#39;s B - your name – iGEM 2015 – date prepared</li>
<li>Make sure that the media has cooled down sufficiently before beginning. However, don&#39;t wait too
long, as the media will start to settle at around 30C.</li>
<li>Take an empty plate and open it slightly. You do not need to open it all the way to pour the agar.
When pouring the agar, pour until 2/3 of the plate has been covered, or approximately half of the
plate has been filled when viewed from the side. Pour the agar slowly, to prevent the formation of
bubbles. Swirl the plate in a circular motion to distribute the media evenly on the plate.</li>
<li>After pouring, set the plates to cool in stacks of 2 or 3, as this takes up less space. Don&#39;t stack the
plates too high – we want to prevent spills.</li>
<li>Rinse the bottle with water before the remnants solidify and become hard to remove.</li>
<li>Allow the plates to cool for at least 20 minutes until the agar has solidified. Flip the plates to prevent
condensation forming on the agar. The plates can then be stacked and stored in plastic bags (ideally re-
use the plastic bags that the plates come in.)</li>
<li>The agar plates should be stored in a 4oC freezer, should be good for 1-2 months</li>
</ol>
<h3 id="kb-leaving-the-lab">KB Leaving the Lab</h3>
<p>Prior to leaving the lab, you should:</p>
<ol>
<li>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</li>
<li>Wipe down your workspace.</li>
<li>Ensure that all materials have been returned to their places, and that the plates have been properly
stored in the fridge.</li>
</ol>
<h3 id="kb-references">KB References</h3>
<p>From: Cold Springs Harbour Protocol. King&#39;s B Medium. From:
<a href="http://cshprotocols.cshlp.org/content/2009/7/pdb.rec11326.full?text_only=true">http://cshprotocols.cshlp.org/content/2009/7/pdb.rec11326.full?text_only=true</a></p>
<p>Recipe for King&#39;s B medium. Received from Maggie Middleton, Lab technician at the David Guttman
Lab.</p>
<h2 id="antibiotic-stock-preparation">Antibiotic Stock Preparation</h2>
<p><em>Written by Tim Lee</em></p>
<h3 id="-background-"><strong>Background</strong></h3>
<p>  The preparation of antibiotic stock is a relatively simple series of mixing and dilutions. The
antibiotic stocks are typically made from four antibiotics : Ampicillin, Kanamycin, Tetracycline
and Chloramphenicol. With the exception of chloramphenicol, the other antibiotics are light
sensitive and once stock is prepared they are wrapped in foil to reduce light exposure.</p>
<p>  An antibiotic comes in a powder form as the basic stock and is mixed with milli-Q water
to form a stock solution. Antibiotic stock preparation follows the same basic process for each
antibiotic, varying only in concentration which is dependent on the type of antibiotic used.</p>
<p>  Antibiotic stocks are prepared for use either in liquid or solid media. In the case of liquid
media, the antibiotic stock is simply added and mixed in. For solid media, usually agar and LB, the
media is sent for autoclaving, and while the media is still hot the antibiotic is added and mixed.
This is to ensure that the antibiotic can be mixed evenly amongst the media before it solidifies into
a solid state.</p>
<p>Antibiotics are used for the purpose of selection. In terms of synthetic biology, plasmids
confer selective antibiotic resistance when successfully transformed into their target bacteria.
Using liquid or solid growth media that has been treated with antibiotics provides a selection
control for those bacteria that have successfully incorporated the plasmid.</p>
<h3 id="-basic-terminology-and-concepts-"><strong>Basic Terminology and Concepts</strong></h3>
<p>There are some terms which will be commonly used to describe the preparation of
antibiotic stocks. Familiarity and understanding of these terms is key to comprehend the protocol.</p>
<p><strong>Milli-Q</strong>- Water that has been purified through successive steps of filtration and
deionization. The standard used in our lab is typically 18.2 MΩ·cm at 25 °C, measured in
resistance due to the lack of ions. The filters used are 0.22 μm in size to ensure a high
level of purity. This water is used for preparing our antibacterial stock to ensure purity.</p>
<p><strong>LB</strong> – Lysogeny Broth, is a very standard and simple media to create due to its recipe
consisting of 3 components - tryptone, yeast extract, and sodium chloride. This mix of
anhydrous ingredients is added to water and then autoclaved, producing liquid media. The
production of solid LB for plates is done by adding agar, a protein isolated from certain
species of seaweed which coagulates the liquid into a gel like form when cooled. This is
done prior to autoclaving and is poured into the plates while still hot, where it will cool
into the plate shape.</p>
<p><strong>Aliquot</strong> - An aliquot is a term to denote a certain quantity of something. In this case an
aliquot of antibiotic stock would denote either 1mL or 0.5mL, depending on the antibiotic.</p>
<p><strong>Antibiotic</strong>- Compounds which inhibit bacterial growth. They act either bacteriostatically
by preventing reproduction of the bacteria, or bacteriocidally where they directly kill the
bacteria. Generally bacteriocides work by interfering with the synthesis of peptidoglycan in
the bacteria&#39;s cell walls. Tetracycline is an example of a bacteriostatic, where it acts by
binding to the ribosomes of prokaryotic bacteria and inhibits translation.</p>
<p><strong>Autoclave</strong>- A piece of equipment used for sterilization. The autoclave performs much like
a pressure cooker: it subjects the contents inside it to a high temperature and high pressure
steam bath. Usually the temperature is 121 Celsius and at 15lbs/in2
20 minutes is enough to kill most microorganisms and render equipment sterile. When
adding antibiotics to media, it is done after autoclaving so that the heat does not destroy
the antibiotic activity.</p>
<h3 id="safety-precautions">Safety Precautions</h3>
<ul>
<li>Be careful when handling the antibiotic, and avoid contact with eyes, mouth and skin</li>
<li>When pouring liquid, commit to the pour to prevent spills.</li>
<li>Use proper PPE</li>
</ul>
<h3 id="materials-and-equipment">Materials and Equipment</h3>
<ul>
<li>100mL Pyrex Bottles</li>
<li>Magnetic Stir Bar</li>
<li>Magnetic Stirrer</li>
<li>Antistatic Weighing Boat</li>
<li>Analytical Balance</li>
<li>Milli-Q Water Dispenser</li>
<li>50mL Falcon Tubes</li>
<li>Aluminum Foil</li>
<li>20mL Syringe</li>
<li>0.22 μm Filter</li>
<li>1mL Microcentrifuge tubes</li>
</ul>
<p><strong>Reagents</strong></p>
<ul>
<li>4g Amphicilin</li>
<li>MilliQ water</li>
<li>400mg tetracycline</li>
<li>80mL 70% EtOH</li>
<li>2.72g chloramphenicol</li>
<li>80mL 100% EtOH</li>
</ul>
<h3 id="procedure">Procedure</h3>
<h4 id="ampicillin">Ampicillin</h4>
<ul>
<li>Stocks and Usage Stock Concentration 50 mg/mL in milliQ water</li>
<li>Aliquots 500μL (use a P1000 set to 0500)</li>
<li>Working Concentration 50 μg/mL
Preparation of 80mL stock solution</li>
</ul>
<p><em>Ampicillin is kept in the 4°C fridge, and is light sensitive. To ensure your stock solution is not
degraded, cover all microcentrifuge tubes used for storing the solution with foil.</em></p>
<ol>
<li>Weigh 4g ampicillin onto an antistatic weighing boat.</li>
<li>Add 80mL milliQ water to a 100mL Pyrex bottle.</li>
<li>Add the ampicillin to the milliQ.</li>
<li>Place a small magnetic stir bar into the solution and place the Pyrex bottle on the stirrer.
Set at 300-600 rpm and stir until dissolved.</li>
<li>Filter sterilise the solution into 50mL Falcon tubes using a 20mL syringe outfitted with a
0.22 μm filter.</li>
<li>Aliquot (500μL) into the appropriate microcentrifuge tubes, labelled with an &quot;A&quot; on top,
and store in the Nalgene racks found in the 20oC.</li>
</ol>
<h4 id="kanamycin">Kanamycin</h4>
<ul>
<li>Stocks and Usage Stock Concentration 10 mg/mL in milliQ water</li>
<li>Aliquots 1 mL (use a P1000 set to 1000)</li>
<li>Working Concentration 50 μg/mL</li>
<li>Add 5 mL of stock per litre of LB. Preparation of 80mL stock solution</li>
</ul>
<p><em>Kanamycin is kept in the 4°C fridge, and is light sensitive. To ensure your stock solution is not
degraded, cover all microcentrifuge tubes used for storing the solution with foil.</em></p>
<ol>
<li>Weigh 800mg kanamycin onto an antistatic weighing boat.</li>
<li>Add 80mL milliQ water to a 100mL Pyrex bottle.</li>
<li>Add the kanamycin to the milliQ.</li>
<li>Place a small magnetic stir bar into the solution and place the Pyrex bottle on the stirrer.
Set at 300-600 rpm and stir until dissolved.</li>
<li>Filter sterilise the solution into 50mL Falcon tubes using a 20mL syringe outfitted with a
0.22 μm filter</li>
<li>Aliquot (1mL) into the appropriate microcentrifuge tubes, labelled with a &quot;K&quot; on top.
Store in the Nalgene racks found in the 20oC. (Make sure microcentrifuge tubes are
covered with foil)</li>
</ol>
<h4 id="tetracycline">Tetracycline</h4>
<ul>
<li>Stocks and Usage Stock Concentration 5 mg/mL in 70% EtOH (N.B. 70% EtOH, not
100% EtOH!)</li>
<li>Aliquots 1mL (use a P1000 set to 1000)</li>
<li>Working Concentration 20 μg/mL
Preparation of 80mL stock solution</li>
</ul>
<p><em>Tetracycline is kept in the 4°C fridge, and is light sensitive. To ensure your stock solution is not
degraded, cover all microcentrifuge tubes used for storing the solution with foil.</em></p>
<ol>
<li>Weigh 400mg tetracycline onto an antistatic weighing boat.</li>
<li>Add 80mL 70% EtOH to a 100mL Pyrex bottle.</li>
<li>Add the tetracycline to the 70% EtOH.</li>
<li>Place a small magnetic stir bar into the solution and place the Pyrex bottle on the stirrer.
Set at 300-600 rpm and stir until dissolved.</li>
<li>Recommended, but optional because of storage in 70% EtOH: Filter sterilise the solution
into 50mL Falcon tubes using a 20mL syringe outfitted with a 0.22 μm filter.</li>
<li>Aliquot into the appropriate microcentrifuge tubes, labelled with a &quot;T&quot; on top. Store in the
Nalgene racks found in the 20oC.</li>
</ol>
<h4 id="chloramphenicol">Chloramphenicol</h4>
<ul>
<li>Stocks and Usage Stock Concentration 34 mg/mL in 100% EtOH (N.B. 100% EtOH, not
70%!)</li>
<li>Aliquots 1 mL (P1000 set to 1000)</li>
<li>Working Concentration 25μL/mL. Preparation of 80mL stock solution</li>
</ul>
<p><em>Chloramphenicol is kept with the general chemicals, and is not light sensitive. The microcentrifuge tubes do not need to be covered with foil to store chloramphenicol.</em></p>
<ol>
<li>Weigh 2.72g chloramphenicol onto an antistatic weighing boat.</li>
<li>Add 80mL 100% EtOH to a 100mL Pyrex bottle.</li>
<li>Add the chloramphenicol to the 100% EtOH.</li>
<li>Place a small magnetic stir bar into the solution and place the Pyrex bottle on the stirrer.
Set at 300-600 rpm and stir until dissolved.</li>
<li>Aliquot (1mL) into the appropriate microcentrifuge tubes, labelled with a &quot;C&quot; on top.
Store in the Nalgene racks found in the 20oC. Note: No filter sterilization is needed
because it is stored in 100% EtOH.</li>
</ol>
<h3 id="leaving-the-lab">Leaving the Lab</h3>
<p>Prior to leaving the lab, you should:</p>
<ul>
<li>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated
individual.</li>
<li>Wipe down your workspace.</li>
<li>Ensure that all materials have been returned to their places, and that the plates have been
properly stored in the fridge.</li>
</ul>
<h3 id="acknowledgement">Acknowledgement</h3>
<p>Protocols of previous iGEM teams were used to make this guideline.</p>
<h2 id="gel-electrophoresis">Gel Electrophoresis</h2>
<p><em>Written by Seray Cicek</em></p>
<h3 id="part-1-casting-the-gel">Part 1: Casting the gel</h3>
<ol>
<li>Add 0.5g of Agarose and 50ml of 1X TAE buffer to an Erlenmeyer flask. (Note that TAE
buffer in the stock has 50X concentration and needs to be diluted with autoclaved Milli-Q
water)</li>
<li>Place the flask into the microwave. <strong>After this step flask will be hot. Use heat
gloves</strong></li>
<li>Set the timer to 2 min. Start the microwave oven but stop every 30 sec to swirl the
contents of the flask. This helps suspend the undissolved agarose. Continue until agarose
particles are dissolved.</li>
<li>While hot, add 3ul of 10,000x SYBR Safe into the agarose solution
and mix throughly.</li>
<li>Set the flask aside for cooling. Occasionally swirl to make the cooling even.</li>
<li>Set up the gel tray and make sure that there is no leak by testing it with dH20 and kimwipes.</li>
<li>When the flask is cool enough for you to handle easily, pour the solution into the tray.</li>
<li>Wait until the gel solidifies.</li>
</ol>
<h3 id="part-2-loading-the-gel">Part 2: Loading the gel</h3>
<ol>
<li>Make a note of what is going into each well and put that in your lab notebook.</li>
<li>Determine the DNA concentrations via Nanodrop. (Approximately 100ng DNA per band
is required.)</li>
<li>Cut a piece of parafilm to use as a mixing surface. Alternatively can use PCR tubes.</li>
<li>Mix DNA with loading dye in a 1:1 ratio directly on the parafilm or in the PCR tubes using a pipette tip and gently aspirating. (i.e. 10uL of DNA with 2uL of 6x loading buffer) Mix by pipetting up and down. Avoid bubbles.</li>
<li>Carefully remove the comb of the gel.</li>
<li>Place the gel into the gel electrophoresis apparatus with its tray.</li>
<li>Add more 1X TAE buffer to cover the gel completely. Buffer should be 1-2mm above
the gel. Make sure that all the wells are submerged.</li>
<li>Load 10uL of DNA ladder to the first well. Add 10uL of your sample DNA
to corresponding wells. (Remember record which wells you load)</li>
<li>Put the lid on. The negative (-/black) electrode should be closer to the DNA wells as DNA
migrates towards the positive (+/red) electrode.
10.Set the rig to 100V &amp; 60 min. Hit start.
11.Make sure that your DNA does not run off the gel. The loading buffer enables you to track
the DNA.
12.The gel apparatus will stop on its own when the time is up. Proceed to imaging the gel. It can sit in the rig overnight if need be.</li>
</ol>
<h3 id="references-">References:</h3>
<p>igem wiki: <a href="http://local.biochemistry.utoronto.ca/igem/index.php/Analytical_Gel_Electrophoresis">http://local.biochemistry.utoronto.ca/igem/index.php/Analytical_Gel_Electrophoresis</a></p>
<p>jove: <a href="http://www.jove.com/video/3923/agarose-gel-electrophoresis-for-the-separation-of-dna-">http://www.jove.com/video/3923/agarose-gel-electrophoresis-for-the-separation-of-dna-</a> fragments</p>
<p>open wetware: <a href="http://openwetware.org/wiki/Agarose_gel_electrophoresis">http://openwetware.org/wiki/Agarose_gel_electrophoresis</a></p>
<p>lifetechno: <a href="https://www.lifetechnologies.com/ca/en/home/life-science/pcr/elevate-pcr-">https://www.lifetechnologies.com/ca/en/home/life-science/pcr/elevate-pcr-</a> research/agarose-content-with-tips-and-tricks.html</p>
<p>Syber green catalog: <a href="http://www.lifetechnologies.com/order/catalog/product/S33102">http://www.lifetechnologies.com/order/catalog/product/S33102</a></p>
<p>Syber green staining protocols: <a href="http://www.lumiprobe.com/protocols/sybr-green-gel-staining">http://www.lumiprobe.com/protocols/sybr-green-gel-staining</a></p>
<h2 id="making-chemically-competent-cells">Making Chemically Competent Cells</h2>
<p><em>Protocol obtained from openwetware:</em> <a href="http://openwetware.org/wiki/Preparing_chemically_competent_cells">http://openwetware.org/wiki/Preparing_chemically_competent_cells</a></p>
<h3 id="materials">Materials</h3>
<ul>
<li>Plate of cells to be made competent</li>
<li>TSS buffer</li>
<li>LB media</li>
<li>Ice</li>
</ul>
<h3 id="glassware-equipment">Glassware &amp; Equipment</h3>
<ul>
<li>Falcon tubes</li>
<li>500μl Eppendorf tubes, on ice</li>
<li>200ml conical flask</li>
<li>200μl pipetman or repeating pipettor</li>
<li>5ml pipette</li>
</ul>
<h3 id="protocol">Protocol</h3>
<h4 id="-part-1-tss-buffer-"><strong>Part 1: TSS Buffer</strong></h4>
<p>To make 50 mL:</p>
<ul>
<li>5g PEG 8000</li>
<li>1.5 mL 1M MgCl2 (or 0.30g MgCl2*6H20)</li>
<li>2.5 mL DMSO</li>
<li>Add LB to 50 mL</li>
<li>Filter sterilize (0.22 μm filter)</li>
</ul>
<h4 id="-part-2-making-chemically-competent-cells-"><strong>Part 2: Making Chemically Competent Cells</strong></h4>
<ol>
<li>Grow a 5ml overnight culture of cells in LB media. In the morning, dilute this culture
back into 25-50ml of fresh LB media in a 200ml conical flask. You should aim to dilute
the overnight culture by at least 1/100.</li>
<li>Grow the diluted culture to an OD600 of 0.2 - 0.5. (You will get a very small pellet if you
grow 25ml to OD600 0.2)</li>
<li>Put eppendorf tubes on ice now so that they are cold when cells are aliquoted into them
later. If your culture is X ml, you will need X tubes. At this point you should also make
sure that your TSS is being chilled (it should be stored at 4
fresh then put it in an ice bath).</li>
<li>Split the culture into two 50ml falcon tubes and incubate on ice for 10 min.
All subsequent steps should be carried out at 4
wherever possible</li>
<li>Centrifuge for 10 minutes at 3000 rpm and 4</li>
<li>Remove supernatant. The cell pellets should be sufficiently solid that you can just pour
off the supernatant if you are careful. Pipette out any remaining media.</li>
<li>Resuspend in chilled TSS buffer. The volume of TSS to use is 10% of the culture volume
that you spun down. You may need to vortex gently to fully resuspend the culture, keep
an eye out for small cell aggregates even after the pellet is completely off the wall.</li>
<li><p>Add 100 μl aliquots to your chilled eppendorfs and store at − 80oC.</p>
<p>o Freeze the cells immediately using a dry ice bath.</p>
<p>o If you run a control every time you clone (i.e. a vector-only ligation), you can as
you can as
well freeze cells in 200 μl aliquots. Unused cells can be frozen back once and
reused, albeit with some loss of competence.</p>
</li>
</ol>
<h2 id="lab-protocol-for-bacterial-transformation">Lab protocol for bacterial transformation</h2>
<p><em>Written by Matt D&#39;Iorio</em></p>
<h3 id="introduction">Introduction</h3>
<p>In this protocol, you will be using chemically competent E coli that have been
placed in a calcium chloride solution prior to freezing. The solution acts to
neutralize the cells so that they don&#39;t repel each other. The bacteria and plasmid
mixture will be chilled on an ice for 30 minutes. Placing the mixture in a 42
degrees Celsius water bath for 30 seconds will heat shock the mixture, which will
cause the transformation. Once the LB is added and mixed with the transformed
bacteria, it can be plated on the LB plate with an antibiotic. Since a lot of the
bacteria will not be transformed after the heat shock, plating E. coli with an
antibiotic will ensure only the transformed E. coli survive.
Separate reagents from materials</p>
<h3 id="basic-terminology-and-concepts">Basic Terminology and concepts</h3>
<p><strong>Transformation</strong> – The process in which the genetic makeup of a cell is changed
by introduction of DNA from the surrounding environment.</p>
<p><strong>Competent E. coli</strong> – E. coli in a state where it can allow the uptake of DNA</p>
<p><strong>Heat Shock</strong> – A sudden increase in temperature used to propel a plasmid into a
bacterium</p>
<h3 id="safety-precautions">Safety precautions</h3>
<p>This is a benign lab protocol, but don&#39;t forget that PPE is necessary at all times.
The E. coli you will be working with will be non-pathogenic but it should still be
handled properly, which means it should not come into contact with your skin or
gloves. If you are removing materials from the -80 degrees Celsius freezer, do not
use your bare hands or regular lab gloves, use gloves designated specifically for
the freezer.</p>
<h3 id="reagents">Reagents</h3>
<ul>
<li>Competent E. coli cells</li>
<li>250 μL of SOC</li>
<li>LB plate supplemented with antibiotics</li>
<li>Materials and equipment</li>
<li>Set of micropipettes and pipette tips</li>
<li>5 x1.5mL – microcentrifuge tube</li>
<li>Ice bath</li>
<li>Thermomixer</li>
<li>Streaker/Glass beads</li>
</ul>
<h3 id="lab-protocol">Lab protocol</h3>
<ol>
<li>Thaw competent E. coli on ice<ul>
<li>Competent cells till be in the -80 degrees Celsius freezer</li>
<li>Do NOT thaw cells by hand, if they are warmed by hand they will
no longer be competent</li>
</ul>
</li>
<li>Mix the cells gently with a pipette tip (do not pipette up and down)</li>
<li>Aliquot 50μL of bacteria into a pre-chilled 1.5mL-microcentrifuge tube.</li>
<li>Add 1-2 μL of the plasmid to the bacterial sample, and mix gently with the
pipette tip (don&#39;t pipette up and down)</li>
<li>Incubate the tube on ice for 30 minutes.</li>
<li>While the bacteria are on ice, ensure the water bath is set to 42°C</li>
<li>Put 250μL of LB in a microcentrifuge tube and place it in the 37°C
incubator to warm up during this time</li>
<li>After the 30 minute incubation period, place the tube in the 42°C water bath
(mixing OFF) for exactly 30 seconds (45 seconds for E.coli DH10 beta)</li>
<li>Place the tube on ice for two (2) minutes
10.Add 250μL of pre-warmed SOC</li>
<li>Shake in 37°C incubator, 300 rpm, for 1-2 hours</li>
<li>Spread 100μL of the transformed bacteria on an LB plate supplemented
with the appropriate antibiotic</li>
<li>Label plate with name, date, and what it is (iGEM 2015 – Transformed E.
Coli – Your Name)</li>
<li>Allow the plates to dry inverted, then place in the 37°C incubator O/N</li>
</ol>
<h3 id="leaving-the-lab">Leaving the lab</h3>
<ul>
<li>Clean dirty glassware, or at least set aside the glassware to be cleaned by a
designated individual.</li>
<li>Wipe down your workspace.</li>
<li>Ensure that all materials have been returned to their places, and that the plates
have been properly stored in the fridge.</li>
<li>Dispose of all disposable tubes and pipette tips used in biohazard containers.</li>
<li>Make sure your plates are labelled and put in a place they can be found.</li>
</ul>
<h2 id="cell-lysis-protocol">Cell Lysis Protocol</h2>
<p><em>Written by Kayla Nemr</em></p>
<p><em>In preparation have the desired cells grown out on plates, with single colonies.</em></p>
<h3 id="day-1">Day 1</h3>
<ol>
<li>Create overnight cultures of each from a single colony in 5mL of LB + antibiotic at 37C and 300rpm</li>
</ol>
<h3 id="day-2">Day 2</h3>
<ol>
<li>label 250mL (or larger) baffled flasks, pre-autoclaved.</li>
<li>Add 50mL of LB + antibiotic</li>
<li>Add 500uL of starter culture to the corresponding flask.</li>
<li>Grow at  37C until the OD is approximately 0.1-0.6</li>
<li>Reduce the temperature to 30C (can go as low as 16C)</li>
<li>At the end of the day, add 1mM IPTG</li>
</ol>
<h3 id="day-3-extraction-of-cell-lysates">Day 3 - Extraction of cell lysates</h3>
<ol>
<li>Pellet 25mL in 50mL falcon tubes at 5000g for 10 min at 4C.</li>
<li><p>Resuspend in 2mL of lysis buffer</p>
<ul>
<li>50mM tris-HCl pH 7.5</li>
<li>50-200mM NaCl</li>
<li>5% glycerol (v/v)</li>
<li>1mM DTT</li>
</ul>
</li>
<li><p>Add 20uL 100mM PMSF</p>
</li>
<li>Add lysozyme to final concentration of 300ug/mL</li>
<li>Incubate tubes on ice horizontally in ice buckets and place on orbiter (for 3 hours)</li>
<li>Transfer to 2mL eppendorf tubes</li>
<li>Pellet in microcentrifuge at 13000g for 10 minutes at 4C.</li>
</ol>
<h2 id="sds-page-protocol">SDS-PAGE Protocol</h2>
<p><em>Written by Kayla Nemr, with Seray Cicek, Sarah Bi, and Roy Lee</em></p>
<h3 id="-introduction-"><strong>Introduction</strong></h3>
<p>Sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS-PAGE) is a technique commonly used in molecular biology which allows for the separation of proteins depending on the size of each protein and helps identify the proteins that are being separated.</p>
<p>Please watch videos shown below to understand the protocol more in depth.</p>
<p>Making the gel: <a href="https://www.youtube.com/watch?v=EDi_n_0NiF4">https://www.youtube.com/watch?v=EDi_n_0NiF4</a></p>
<p>Running the gel: <a href="https://www.youtube.com/watch?v=XUjLO-ek2C8">https://www.youtube.com/watch?v=XUjLO-ek2C8</a></p>
<p>Staining the gel: <a href="https://www.youtube.com/watch?v=b-1dXzU4iOw">https://www.youtube.com/watch?v=b-1dXzU4iOw</a></p>
<h3 id="-basic-terminology-and-concepts-"><strong>Basic Terminology and Concepts</strong></h3>
<p><strong>Protein ladder:</strong> a premade solution containing proteins of known sizes. Analyzing the electrophoresis alongside the ladder makes it easier to identify the various sizes of the fragments in the tested protein samples by comparing them to the known protein sizes of the ladder.</p>
<p><strong>Resolving gel:</strong> this section of the gel will be the section that separates the proteins based on weight.</p>
<p><strong>Stacking gel:</strong> this section of the gel will be the section where the protein samples are loaded. This section does not separate the proteins based on weight and after the electrophoresis is complete, provides no information and may be thrown out.</p>
<p><strong>Buffer:</strong> the buffer allows for current to run through the gel Materials and Equipment</p>
<h3 id="-materials-"><strong>Materials</strong></h3>
<ul>
<li>6.04g of Tris base</li>
<li>28.8g of glycine</li>
<li>2g SDS</li>
<li>ddH2O</li>
<li>1.5 M Tris, pH 8.8</li>
<li>1.0 M Tris, pH 6.8</li>
<li>30% bis-acrylamide</li>
<li>10% SDS</li>
<li>10% APS</li>
<li>TEMED</li>
<li>30% Isopropanol</li>
<li>Methanol</li>
<li>Coomassie Brilliant Blue R250</li>
<li>100% ethanol</li>
<li>Glacial acetic acid</li>
<li>Protein samples</li>
<li>Protein ladder</li>
<li>SDS loading dye 6x</li>
</ul>
<h3 id="equipment">Equipment</h3>
<ul>
<li>Micropipette and tips</li>
<li>Glass plates</li>
<li>Casting frame</li>
<li>Casting stand</li>
<li>Gel comb</li>
<li>Tweezers/spatula</li>
<li>Filter paper/nitrocellulose membrane</li>
<li>Erlenmeyer flasks</li>
<li>Electrophoresis apparatus (Gel apparatus/Cassette/Tank)</li>
<li>Power supply</li>
<li>Wide containers</li>
</ul>
<h3 id="lab-protocol">Lab Protocol</h3>
<h4 id="-preparation-of-1x-sds-page-running-buffer-"><strong>Preparation of 1X SDS-PAGE Running Buffer.</strong></h4>
<p><em>This recipe makes 2L of 1X SDS-PAGE running buffer.</em></p>
<p><strong> CAUTION </strong>  - SDS powder is hazardous. Prepare solution in a ventilated fume hood.</p>
<ul>
<li>Dissolve 6.04g of Tris base and 28.8g of glycine together in 1.8 L of ddH2O.</li>
<li>Add 2g SDS and mix.</li>
<li>Add ddH2O to a final volume of 2 L.</li>
</ul>
<h4 id="-preparation-of-sds-page-gel-"><strong>Preparation of SDS-PAGE gel</strong></h4>
<ul>
<li>Insert glass plates (0.75 mm) into the casting frame and secure them into place. Make sure that the shorter</li>
<li>plate is in front and that the bottom is flat.</li>
<li>Put the casting frame on the casting stand.</li>
<li>Insert the comb in between the glass plates. Take a marker and make a mark approximately 1</li>
<li>cm beneath the comb. Take the comb out.</li>
<li>If necessary, test the leakiness of the apparatus by adding water. Draw out the water using filter</li>
<li>paper.</li>
<li>Start making the resolving gel in a separate flask</li>
</ul>
<p>The recipe for the <strong>resolving/separating gel</strong> (10mL, 12%) is as follows:</p>
<p><em>This will create 10 mL of gel. Around &gt;3.5 mL is needed for each gel.TEMED must be added last and the gel should be casted immediately after its addition.</em></p>
<ul>
<li>3.35mL of dH2O (milliQ)</li>
<li>2.5 mL of 1.5 M Tris, pH 6.8</li>
<li>100 uL of 10% SDS</li>
<li>4mL of 30% bis-acrylamide</li>
<li>50 uL of 10% Ammonium persulfate (APS)</li>
<li><p>5 uL of TEMED</p>
<p>Add gel into the apparatus until it reaches the previously created mark</p>
</li>
<li><p>Remove any air bubbles by adding a layer of isopropanol on top of the gel. Wait until the</p>
</li>
<li>resolving gel has polymerized (usually takes 30-45 minutes) and remove the isopropanol by pouring it or by using filter paper.</li>
<li>Start making stacking gel in a separate flask</li>
</ul>
<p>The recipe for the <strong>stacking gel</strong> (10mL, 4%) is as follows:</p>
<p><em>This will create 10 mL of gel. Around 1.2 mL is needed for each gel. TEMED must be added last and the gel should be casted immediately after its addition.</em></p>
<ul>
<li>7.3 mL of dH20</li>
<li>1.25 mL of 1.0 M Tris, pH 6.8</li>
<li>100uL of 10% SDS</li>
<li>1.34 mL of 30% bis-acrylamide</li>
<li>50uL of 10% APS</li>
<li>10uL of TEMED</li>
<li><p>Fill the rest of the apparatus with stacking gel and place the gel comb in. Ensure that there are</p>
<p>no air bubbles. Wait until the gel solidifies (around 30-45 minutes).</p>
</li>
</ul>
<h4 id="-running-the-sds-page-gel-"><strong>Running the SDS-PAGE gel</strong></h4>
<ol>
<li>Take out the gel (with the plates) from the casting apparatus and place it in the gel apparatus.</li>
<li>Make sure that the short plate faces inwards to the apparatus. If only one gel is being run, make</li>
<li>sure to close the other side of the apparatus with a dummy plate.</li>
<li>Once the two plates are secured, insert them into the cassette and lock the apparatus in place.</li>
<li>Insert the cassette into the tank.</li>
<li>Prepare the samples. Add 3uL of 6x sample buffer to 15uL of sample, then boil the samples. This can be done by setting a thermocycler to 95C for 10 minutes.</li>
<li>Meanwhile, fill the inner chamber (in between the glass plates) with buffer until the buffer</li>
<li>reaches the top of the inner chamber. Fill the bottom of the outer chambers with more buffer</li>
<li>so that it covers the bottom of the gels by at least 2cm. A large amount is not needed. Take the</li>
<li>gel comb out gently. straighten any lopsided well walls with</li>
<li>pipette tip.</li>
<li>Add the protein ladder to the first well and the protein samples in the other</li>
<li>wells. Load 10-20ul per well (same volume for every well). Be careful not to overfill a well!</li>
<li>Record order of loading in lab notebook!</li>
<li>Dip the pipette tip through the surface of the buffer, centre it near the bottom of the well and</li>
<li>SLOWLY depress the pipette plunger into the well. Make sure all of the solution actually goes</li>
<li>into the well as to not contaminate the other wells.</li>
<li>Place the lid on the apparatus and connect the apparatus to the power supply.</li>
<li>Make sure the cathode and anode are connected correctly (black to black, red to red)</li>
<li>Set the voltage to 140 and run the gel for around 70 minutes.</li>
<li>Monitor movement of loading dye through gel. Stop running when dye front reaches bottom of</li>
<li>gel. Once the process is finished, take off the lid, remove the inner chamber and pour out the</li>
<li>buffer carefully.</li>
<li>Remove the glass carefully.</li>
<li>Rinse the plates with water and separate the plates</li>
<li>The stacking gel should be removed by cutting with a spatula or razor blade. Try not to tear the</li>
<li>resolving gel.</li>
</ol>
<h4 id="-staining-the-sds-page-gel-"><strong>Staining the SDS-PAGE gel</strong></h4>
<ol>
<li>Fill a wide container with deionized water and separate the resolving gel from the glass plate. Leave the</li>
<li>gel in the container.</li>
<li>Rinse the gel with deionized water gently. Repeat 2-3 times.</li>
<li>In a container with deionized water, microwave the gel for 30 seconds.</li>
<li>Transfer the gel to the staining solution and microwave for 30 seconds.</li>
</ol>
<p><strong>To make the staining solution:</strong></p>
<ul>
<li>In a new container, add 2.5 g of Coomassie Brilliant Blue R250 to 450mL 100% ethanol. Dissolve</li>
<li>the Brilliant Blue with a magnetic stir bar.</li>
<li>Add 100 mL of glacial acetic acid</li>
<li>Add 450 mL dH2O</li>
<li>Optional: filter through Whatman No.1 filter</li>
<li>Leave the gel on an orbital shaker for 20-60 minutes, until strong bands are visible.</li>
</ul>
<p><strong>To make the staining solution</strong></p>
<ul>
<li><p>In a new container, add 800 mL of 100% ethanol and 400 mL of glacial acetic acid to 2800 mL of</p>
<p>dH2O (destaining solution)</p>
</li>
<li><p>Transfer the gel to the new destaining solution. Add kimwipes to aid in the destaining process.</p>
</li>
</ul>
<h3 id="-leaving-the-lab-"><strong>Leaving the lab</strong></h3>
<ul>
<li>Ensure that the gels are disposed of in the proper waste bins</li>
<li>Dispose of any excess buffer according to regulations</li>
<li>Disconnect power supply and ensure that all machinery is turned off</li>
</ul>
<h3 id="references">References</h3>
<p>Protocol for separating and stacking gels, sample preparation and gel staining obtained from Kayla Nemr.</p>
<p>Preparing gels: <a href="http://www.ruf.rice.edu/~bioslabs/studies/sds-page/gellab2a.html">http://www.ruf.rice.edu/~bioslabs/studies/sds-page/gellab2a.html</a></p>
<p>Assembling, loading, running gels: <a href="http://www.ruf.rice.edu/~bioslabs/studies/sds-page/gellab2b.html">http://www.ruf.rice.edu/~bioslabs/studies/sds-page/gellab2b.html</a></p>
<p>Jove: <a href="http://www.jove.com/science-education/5058/separating-protein-with-sds-page">http://www.jove.com/science-education/5058/separating-protein-with-sds-page</a></p>
<h2 id="purelink-pcr-purification-protocol">PureLink® PCR Purification protocol</h2>
<p><em>Minor changes were made to the protocol supplied with the Invitrogen PureLink® PCR Purification Kit</em></p>
<ol>
<li><p>Add 4 volumes of PureLink
®
Binding Buffer (B2)
with isopropanol to
1
volume of
the PCR product (50–100 μL).
Mix well.</p>
</li>
<li><p>Remove a PureLink
®
Spin Column in a Collection
Tube from the package.</p>
</li>
<li><p>Add the sample with the appropriate Binding
Buffer (from step 1 of this procedure) to the
PureLink
®
Spin Column.</p>
</li>
<li><p>Centrifuge the column at room temperature at
13,000 ×
g
for 1 minute.</p>
</li>
<li><p>Discard the flow through and place the spin
column into the collection tube.</p>
</li>
<li><p>Add 650 μL of Wash Buffer with ethanol to the column.</p>
</li>
<li><p>Centrifuge th
e column at
room temperature at
13,000 ×
g
for 1 minute. Discard the flow through
from the collection tube and place the column
into the tube.</p>
</li>
<li><p>Centrifuge the column at maximum speed (15, 000 x g)at
room temperature for 2–3
minutes to remove any
residual Wash Buffer. Discard the collection tube.</p>
</li>
<li><p>Place the spin column in a clean 1.7-mL
PureLink
®
Elution Tube supplied with the kit.</p>
</li>
<li><p>Add 50 μL of Elution Bu
ffer (10 mM Tris-HCl,
pH 8.5) or sterile, distilled water (pH &gt;7.0) to the
center of the column.</p>
</li>
<li><p>Incubate the column at room temperature for
1 minute.  Centrifuge the column at maximum speed for
2 minutes.</p>
</li>
<li><p>The elution tube contains the purified PCR product.
Remove and discard the column. Store the purified PCR product at –20°C or use
the PCR product for the desired downstream
application.</p>
</li>
</ol>
<h3 id="reference">Reference</h3>
<p>PureLink® PCR Purification protocol. From: <a href="http://tools.thermofisher.com/content/sfs/manuals/purelink_pcr_man.pdf">http://tools.thermofisher.com/content/sfs/manuals/purelink_pcr_man.pdf</a></p>
<h2 id="quantum-prep-plasmid-miniprep-protocol">Quantum Prep Plasmid Miniprep Protocol</h2>
<p><em>Minor changes were made to the protocol supplied with the Bio-Rad Quantum Prep Plasmid Miniprep Kit</em></p>
<p>All centrifugation steps are performed at maximum speed
(12,000–14,000 x g) unless otherwise stated.</p>
<ol>
<li><p>Transfer an overnight culture (1–2 ml) of plasmid-containing
cells to a microcentrifuge tube. Pellet the cells by centrifugation
for 15–30 secs. If the overnight culture is larger than 2mL, centrifuge the falcon tubes at 3000g for 10 minutes and proceed. Remove all of the supernatant by pipeting.</p>
</li>
<li><p>Add 200 μl of the cell resuspension solution and vortex until the cell pellet is completely resuspended.</p>
</li>
<li><p>Add 250 μl of the cell lysis solution and mix by gently inverting
the capped tube about ten times (do not vortex). The solution should become viscous and slightly clear if cell lysis has occurred.</p>
</li>
<li><p>Add 250 μl of the neutralization solution and mix by gently
inverting the capped tube about ten times (do not vortex). A
visible precipitate should form.</p>
</li>
<li><p>Pellet the cell debris for 5 mins in a microcentrifuge. (If used falcon tubes centrifuge for the same amount of time.) A
compact white debris pellet will form along the side or at the
bottom of the tube. The supernatant (cleared lysate) at this
step contains the plasmid DNA.</p>
</li>
<li><p>While waiting for the centrifugation step at step 5, insert a spin
filter into one of the 2 ml microcentrifuge wash tubes supplied
with the kit. Mix the Quantum Prep matrix by vortexing or
repeated shaking and inversion of the bottle to insure that it is
completely suspended.</p>
</li>
<li><p>Transfer the cleared lysate (supernatant) from step 5 to a spin
filter. Split the total volume (700uL) into two, adding 350uL of lysate into each filter. Add 100 μl of thoroughly suspended Quantum Prep
matrix to each, then pipet up and down to mix. If you have multiple
samples, transfer the lysates first, then add matrix and mix.
When matrix has been added to all samples and mixed,
centrifuge for 30 secs.</p>
</li>
<li><p>Remove the spin filter from the 2 ml tube, discard the filtrate at
the bottom of the tube, and replace the spin filter in the same
tube. Add 250 μl of wash buffer and wash the matrix by
centrifugation for 30 seconds.</p>
</li>
<li><p>Remove the spin filter from the 2 ml tube, discard the filtrate
at the bottom of the tube and replace the spin filter in the
same tube. Add 250 μl of wash buffer and wash the matrix
by centrifugation for a full
2 mins
to remove residual traces
of ethanol.</p>
</li>
<li><p>Remove the spin filter and discard the microcentrifuge tube. Place
the spin filter in one of the 1.5 ml collection tubes supplied with
the kit or in any standard 1.5 ml microcentrifuge tube that will
accomodate the spin filter. Add 50 μl of nuclease free H
2
O or TE.
Elute the DNA by centrifugation for 1 min at top speed.</p>
</li>
</ol>
<ol>
<li>Discard the spin filter and store the eluted DNA at -20°. Make sure to label properly.</li>
</ol>
<h3 id="reference">Reference</h3>
<p>Bio-Rad Quantum Prep Plasmid Miniprep Kit Instruction Manual. From: <a href="http://www.bio-rad.com/webroot/web/pdf/lsr/literature/MS4100066F.pdf">http://www.bio-rad.com/webroot/web/pdf/lsr/literature/MS4100066F.pdf</a></p>
<h2 id="pcr-based-rdp-parts-creation-protocol">PCR-Based RDP Parts Creation Protocol</h2>
<p><em>Written by Synbiota</em></p>
<h3 id="introduction">Introduction</h3>
<p>This protocol describes how to create RDP parts larger than 100bp using PCR. To create parts &lt;100bp in length, refer to the Oligo-Based RDP Parts Creation Protocol.</p>
<h3 id="buffers-reagents-and-consumables">Buffers, Reagents and Consumables</h3>
<p><strong>Supplied</strong></p>
<ul>
<li>GeneRuler DNA Ladder Mix (Thermo Scientific)</li>
<li>TE Buffer</li>
<li>LM (Load Mix)</li>
</ul>
<p><strong>Not supplied</strong></p>
<ul>
<li>Phusion® High-Fidelity DNA Polymerase and its associated 5x HF buffer</li>
<li>dNTP mix (10mm)</li>
<li>Forward and reverse PCR primers for your RDP part</li>
<li>Qiagen Quick cleanup PCR spin-columns and reagents (PB and PE buffers)</li>
<li>TAE stock buffer (Tris-Acetate-EDTA @ 2M ) in deionized H<sub>2</sub>O</li>
<li>Agarose</li>
<li>DNA stain (SYBR Green or RED Safe)</li>
</ul>
<h3 id="pcr-amplification">PCR Amplification</h3>
<p><strong>Important:</strong> The following protocol has been optimized using Phusion® DNA polymerase. <strong>We strongly recommend you use Phusion polymerase for this protocol.</strong></p>
<ol>
<li><p>Primer Preparation</p>
<p> Upon receipt of your RDP part primers, briefly centrifuge each tube to bring the solid material to the tube bottom. Resuspend each primer to a final concentration of 25nm/mL (25μM) in TE buffer using the formula:</p>
<p> <strong>Resuspension volume (&#956;L) = nm (from supplier label) x 40</strong></p>
<p> Vortex at maximum setting for 10 seconds, let stand 10 min then repeat twice. Briefly centrifuge and freeze until needed.</p>
<p> <strong>Note:</strong> Keep track of your PCR primers, since you may be able to use some of them to sequence RDP parts in assembled circuits. Store frozen.</p>
</li>
<li><p>Template preparation</p>
<p> If the starting template for the final RDP part is contained within a circular vector such as a plasmid, then linearize it by choosing an appropriate restriction endonuclease whose site is located in the vector backbone but not within the part sequence. PCR amplification from a linear template increases the final yield of the PCR product while reducing the level of lowerband contaminants.</p>
<p>Since the PCR reaction uses 2μL of template @10ng/μL, digest 100ng of DNA in a final volume of 10μL. Add 2μL of the digest to the PCR reaction just prior to amplification as described below.</p>
</li>
<li><p>Reaction setup</p>
<p> The parts creation protocol requires a minimum of 5μg of PCR product that should be easily obtained from a 100μL reaction as described below:</p>
<p> Keep all reagents on ice. Using a 0.2mL PCR tube, combine in the following order:</p>
<ul>
<li>2&#956;L forward primer</li>
<li>2&#956;L reverse primer</li>
<li>20&#956;L 5x HF Buffer</li>
<li>dNTPs (@10mM)</li>
<li>65&#956;L H<sub>2</sub>O</li>
<li>2&#956;L template</li>
<li><p>1&#956;L Phusion polymerase (@2u/μL)</p>
<p>Mix the reaction by pipetting up and down gently to avoid bubbles. Centrifuge (pulse) briefly. Then keep on ice until amplification.</p>
</li>
</ul>
</li>
<li><p>PCR Amplification</p>
<p>Amplify using the cycling program shown below:</p>
<ol>
<li>Intitial denaturation at 98C for 30s</li>
<li>Denaturation at 98C for 10s</li>
<li>Annealing at 64C for 45s</li>
<li>Extension at 72C for 30s/1kb (determined by multiplying the length of the template (in kilobases) by 30s)</li>
<li>Repeat steps 2-4 for 25 cycles</li>
<li>Final extension at 72C for 10min</li>
<li>Hold at 4C</li>
</ol>
</li>
<li><p>Cleanup</p>
<p> Use the standard Qiagen spin-column cleanup procedure (or equivalent), noting the procedural modifications described below. <strong>Do not</strong> use the spin columns provided with the RDP kit. They have a special cleanup role following the BsaI digestion step.</p>
<ul>
<li>When the Qiagen PB buffer mix is added to the column, let stand one minute before centrifuging (or filtration if using a vacuum manifold).</li>
<li>Following the Qiagen PE buffer wash step, elute the DNA from the column into a 1.5mL microfuge tube using 50μL TE buffer, then repeat with a second 50 μL elution into the same collection tube yielding a final volume of ~85-90μL.</li>
</ul>
</li>
<li><p>Determining the quality and quantity of the PCR Product</p>
<p> For a more accurate measure, use a NanoDrop spectrophotometer. Use 2 μL of the recovered sample. Remember to zero the instrument using the same TE buffer that was used to elute your sample.</p>
<p> If a NanoDrop spectrophotometer is not available, use the procedure below.</p>
<p> <strong>Product size and purity</strong></p>
<ul>
<li>For products with an expected size of 0.2-10 kb, set up a 1% agarose gel using a 1:50 dilution of 50x TAE stock buffer (Tris-Acetate-EDTA @ 2M ) in deionized H<sub>2</sub>O for both the gel and running buffer.</li>
<li><p>To prepare a gel sample of your PCR product, add 1μL of the DNA to 9μL of load mix (LM).</p>
<p><strong>Note:</strong> In order to minimize the interference of dyes with subsequent band quantification, only bromophenol blue has been included at considerably lower than standard concentrations to mitigate the masking of bands in the 300-600 bp range. Store remainder of PCR product at 4C</p>
</li>
<li>Selecting two of the most central wells of the gel, load 10μL of the provided DNA markers (ladder) and the full 10μL sample prepared in step ii) above. A gel figure of the DNA ladder, including band sizes and amounts (ng) is provided below.</li>
<li><p>Following electrophoresis, stain the gel for 15 min with ethidium bromide, SYBR Green, or RED Safe, using whichever method is standard to your lab.</p>
<p><strong>Note:</strong> A high quality PCR product will yield a bright band of the expected size, with only faint traces of unspecific product contaminants. If the product band has an intensity that is at least as intense as the 500 bp,1000 bp, or 3000 bp ladder bands (60 ng/band; select the one that is closest to your product size for comparison), then the PCR yield is at least 60 μg/μL, an acceptable concentration for further steps. If not, then more PCR reactions may be required.</p>
</li>
</ul>
</li>
</ol>
<h3 id="references">References</h3>
<p>Protocol obtained from Synbiota: <a href="https://synbiota.com/beta">https://synbiota.com/beta</a></p>
<h2 id="bsai-digestion-protocol">BsaI Digestion Protocol</h2>
<p><em>Written by Synbiota</em></p>
<h3 id="introduction">Introduction</h3>
<p>The purpose of BsaI Digestion Protocol is to create RDP parts larger than 100bp using PCR. If you desire to create less than &lt;100bp in length, refer to the Olgio-Based RDP Parts Creation Protocol</p>
<h3 id="safety-precautions">Safety Precautions</h3>
<p><strong>SDS (safety data sheet)</strong>: Refer to the SDS sheets for all listed materials before entering the lab. Be
prepared to answer any questions regarding the information on these sheets.</p>
<p><strong>PPE (Personal protective equipment)</strong>: Proper lab attire should be worn throughout the experiment: This
means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact
lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times,
and long hair should be tied back.</p>
<p><strong>Autoclave</strong>: The autoclave should only be handled by execs. <strong>Note that any autoclaved materials
may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</strong></p>
<h3 id="reagents">Reagents</h3>
<ul>
<li>GeneRuler DNA LAdder Mix (Thermo Scientific)</li>
<li>TE Buffer</li>
<li>LM (Load Mix)</li>
<li>BasI-HF restriction endonuclease with accompanying 10x CutSmart Buffer</li>
<li>TAE stock buffer (Tris-Acetate-EDTA @ 2M) in deionized H<sub>2</sub>O</li>
<li>Agarose powder</li>
<li>DNA strain (e.g., ethidium bromide, SYBR Green or RED Safe)</li>
<li>Cleaned up PCR product of RDP part DNA</li>
<li>Qiagen Quick cleanup PCR reagents (PB and PE buffers)</li>
</ul>
<h3 id="materials">Materials</h3>
<ul>
<li>Synbiota micro spin-columns</li>
<li>Qiagen Quick cleanup PCR spin-columns</li>
</ul>
<h3 id="lab-protocol">Lab Protocol</h3>
<h4 id="part-1-bsai-digestion">Part 1: BsaI Digestion</h4>
<ol>
<li>Add 10&#956;L of NEB CutSmart Buffer and 2&#956;L of BsaI-HF (40 units) to cleaned up DNA</li>
<li>Mix and centrifuge briefly to pull down any splashes and eliminate any surface bubbles</li>
<li>Incubate at 37<sup>o</sup>C for 1-4 hours</li>
</ol>
<h4 id="part-2-dna-cleanup">Part 2: DNA Cleanup</h4>
<p>This is for inactivating and removing BsaI and the small flanking fragments that are released on cleavage</p>
<ol>
<li>Add 0.5mL of DNA binding buffer (e.g., Qiagen PB) and vortex</li>
<li>Place a Synbiota spin column into a 1.5mL microfuge tube, then load the entire sample and let stand for one minute.</li>
<li>Centrifuge for 1 minute at 14K rpm then discard the flow-through</li>
<li>Add 200&#956;l of was buffer (e.g., Qiagen PE) to column</li>
<li>Centrifuge for 1 minute at 14K RPM, then discard flow-through</li>
<li>Repeat wash step and discard flow-through</li>
<li>Place column into a clean microfuge tube and add 100&#956;L of Elution Buffer (e.g., Qiagen EB). Let stand for 1 minute then centrifuge for 1 minute at 14K rpm. <strong>DO NOT DISCARD THE COLUMN. It can be reused for steps 8 and 9</strong></li>
<li>Repeats steps 1 to 3 by adding 0.5mL of PB to the DNA eluted in steps 7</li>
<li>For the final elution step, add 25&#956;L of TE buffer to column and let stand for 1 minute then centrifuge for 1 minute. Repeat the elution with another 25&#956;L collected in same tube</li>
</ol>
<h4 id="part-3-measuring-and-adjusting-the-part-concentration">Part 3: Measuring and adjusting the part concentration</h4>
<p>In the RDP assembly protocol, PCR parts are used at a concentration of 0.04 pm/&#956;L per reaction, for a total of 0.2pm in 5&#956;L. Concentrations of 0.02pm will work but doubling the ligation time is advised. Concentrations greater than 0.04pm pose no problem and may be advantageous. However, adding more means less sample for subsequent builds.</p>
<ol>
<li>If have access to Nano Drop or equivalent, first determine the concnetrations of your part in ng/&#956;L using 2&#956;L of recovered sample. Zero the instrument using the same TE buffer that was used to elute your sample</li>
<li>Determine the remaining volume of your sample</li>
<li>To convert the concentrations of your part from ng/&#956;L to pm/&#956;L (pm/&#956;L<sub>Part</sub>), use the very close approximation: 1pm/&#956;L = 670 ng/&#956;L divided by the length of the part (kb)</li>
<li><p>If the concnetrations significantly exceeds 0.04pm/&#956;L, then adjust the recovered volume of your sample (&#956;L<sub>recovered</sub>) with TE buffer using the formula below:</p>
<p>   Add vol (&#956;L) = [25 x pm/&#956;L<sub>Part</sub> x &#956;L<sub>recovered</sub>] - &#956;L<sub>recovered</sub></p>
<p> An alternative method to calculations is a gel-based method and can be used to approximate a workable concentrations as described below:</p>
</li>
<li><p>Set up an agarose gel as described above, and prepare a gel sample of the digested DNA, add 1&#956;L of DNA to 9&#956;L of load mix (LM)</p>
</li>
<li>Load the entire sample into a centrally located well</li>
<li>Load 10&#956;L of DNA ladder to one side of your sample and 15&#956;L to other side</li>
<li>Following electrophoresis, photograph the gel at lowest possible exposure where all bands remain visible but where gel background appears as dark as possible</li>
<li><p>To determine the minimum concentrations of your sample, compare your product band to the nearest sized ladder band in either or both ladder lanes that has an intensity that is closest or lower than yours</p>
<p> NOTE: For the 10&#956;L load, all bands correspond to 20ng of DNA except for the 500, 1000 and 3000 bp bands, which correspond to 60ng. For the 15&#956;L load, all abnds are 30ng of DNA except for the 500, 1000 and 3000 bp bands which are at 90ng.</p>
</li>
<li><p>The value that you decidde on represents the minimum concentration of your sample expressed as &#956;g/&#956;L</p>
</li>
<li>Use formula described in step (3) to convert the concentrations to pm/&#956;L</li>
<li>If minimum concentration of your sample significantly exceeds 0.04 pm/&#956;L then dilute it as described in a section step (4)</li>
<li>Store the remaining RDP Part DNA at ~4<sup>o</sup>C and remember to centrifuge it briefly before each use</li>
</ol>
<h4 id="part-4-part-validation">Part 4: Part Validation</h4>
<p>To determine how well your park works in RDP assembly, incorporate your part into a simple RDP assembly that begins with an Anchor part and ends with a Cap part.</p>
<p>If your part is in the X-Y&#39; format:</p>
<ul>
<li>Start with <strong>dA<sub>20</sub>-ChlrR-X&#39;</strong></li>
<li>Finish with <strong>Z-Ori.3-dT<sub>20</sub></strong></li>
</ul>
<p>If your part is in the Z-X&#39; format:</p>
<ul>
<li>Start with <strong>dA<sub>20</sub>-ChlrR-Z&#39;</strong></li>
<li>Finish with <strong>X-Ori.3-dT<sub>20</sub></strong></li>
</ul>
<h3 id="leaving-the-lab">Leaving the Lab</h3>
<p>Prior to leaving the lab, you should:</p>
<ol>
<li>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</li>
<li>Wipe down your workspace.</li>
<li>Ensure that all materials have been returned to their places, and that the plates have been properly
stored in the fridge.</li>
</ol>
<h3 id="references">References</h3>
<p>Protocol obtained from Synbiota: <a href="https://synbiota.com/beta">https://synbiota.com/beta</a></p>
<h2 id="protocol-for-synbiota-rdp-assembly">Protocol for Synbiota RDP Assembly</h2>
<p><em>Written by Synbiota</em></p>
<h3 id="preparation">Preparation</h3>
<ol>
<li><p>Obtain and refigerate the following buffers:</p>
<ul>
<li>RDP Wash Buffer (10mL)</li>
<li>RDP Elution buffer (10mL)</li>
<li>RDP Storage buffer (10mL)</li>
</ul>
<p><strong>Important:</strong> all liquid reagents stored refrigerated in microcentrifuge tubes will evaporate overtime and condense on the lid. All microcentrifuge tubes should be pulsed down and mixed just prior to use.</p>
</li>
<li><p>Identify the RDP parts you need from the iGEM kit plate.</p>
<p> Based on your design, detrmine the RDP parts you need from the kit.</p>
<p> <strong>Important:</strong> If your assembly has more than 5 RDP parts, include the X-blockers (Blk-X) and Z-blockers (Blk-Z). They aren&#39;t required as part of the circuit, but are needed for the assembly protocol as described further below.</p>
</li>
<li><p>To ensure you resuspend the correct wells, use a Sharpie or lab marker to draw a ring around the upper lip of each selected well</p>
</li>
<li><p>Resuspend RDP parts in high-grade H<sub>2</sub>O.
Use the table provided for resuspension volumes.</p>
<p> For all “non-anchor” parts, there is a purple or yellow pellet of varying size and intensity at the bottom of each well. This is an indicator dye to show that the sample is present. Gently suspend the pellet by pipetting up and down a few times until the solution appears uniformly pink.</p>
<p> Anchor parts are provided pre-bound to the magnetic beads and appear as a dark brown/black pellet in dried form. Resuspension (as above) will result in an opaque brown solution that will settle over time. Make sure the mixture is well mixed just prior to each use.</p>
</li>
<li><p>Store RDP parts</p>
<p> Transfer each resuspended part to a microfuge tube that has been clearly labelled (screw-capped tubes are preferred since they minimize evaporation).</p>
<p> <strong>Refrigerate until needed. Secure the lid tightly on the plate and store refrigerated.</strong></p>
</li>
<li><p>Prepare the ligase solution just prior to use and store on ice.</p>
<p> Each coupling reaction uses 5 μL of T4 DNA ligase at a final concentration of 0.2 Weiss units/μL in 1x ligase buffer. To calculate the volume (in μL) of ligase required, multiply the number of ligation steps in your build by 5 and prepare accordingly. NOTE: ThermoFisher ligase uses Weiss units NEB and Invitrogen use cohesive end units. 1 Weiss unit= ~70 CE units.</p>
</li>
</ol>
<h3 id="rdp-assembly">RDP Assembly</h3>
<p><strong>NOTE:</strong> Before beginning, briefly centrifuge all parts and solutions contained in microfuge tubes and mix thoroughly. Tubes that have sat in the fridge for a while will evaporate and condense liquid up at the cap. This step restores the original volume and concentration of the sample.</p>
<ol>
<li><p>Consult the circuit design. Post a copy of your RDP circuit design so you know the right order of assembly.</p>
</li>
<li><p>Arrange your RDP parts. Put your DNA tubes in the right order.</p>
</li>
<li><p>Begin the assembly by gently mixing the bead-anchor stock solution by pipetting it slowly up and down (avoiding bubbles) until homogenous.
This method minimizes the splashes of other more vigourous methods that strand droplets up the side of the tube.</p>
</li>
<li><p>Immediately transfer 5 μL of the mixture to a 1.5 mL microfuge tube.</p>
</li>
<li><p>Add RDP Wash buffer (WB). Add 50 μL of RDP Wash buffer and mix as above. Pulse the liquid down briefly in a low speed microfuge.</p>
</li>
<li><p>Place the tube in a magnet cradle and wait for the solution to clear, by allowing the beads to collect on the tube wall next to the magnet.
Wait 20-30 seconds until the solution fully clears.</p>
</li>
<li><p>Remove and discard liquid.
Place the pipette tip at the bottom of the tube at the opposite side from the bead pellet. Remove all liquid, including any droplets on side of tube and discard.</p>
</li>
<li><p>Perform a second wash by repeating steps 5-7.</p>
</li>
<li><p>Add first RDP part and ligation master mix.
Add 5 μL of the first part in your design to the bead pellet, then add 5 μL of the ligation master mix. Mix gently as described above taking care to avoid splashing.</p>
</li>
<li><p>Incubate for 8 minutes at room temperature.</p>
</li>
<li><p>Add 5&#956;L of the appropriate Blocker to the existing reaction:</p>
<ul>
<li>Blk-X, if you just added an X-Z’ part.</li>
<li><p>Blk-Z, if you just added a Z-X’ part.</p>
<p><strong>NOTE:</strong> Adding the wrong Blocker will kill the assembly.</p>
</li>
</ul>
</li>
<li><p>Mix gently and incubate an additional 2 minutes.</p>
</li>
<li><p>Repeat steps 3 through 8.</p>
</li>
<li><p>Mark off the coupled part on your circuit diagram.
On your sketch, cross off the part you just coupled so you can keep track of what step you are on.</p>
</li>
<li><p>Repeat steps 9 through 14 for each remaining part, finishing with the Cap.</p>
</li>
<li><p>Resuspend the bead pellet in 20 μL of RDP Elution buffer (EB).</p>
</li>
<li><p>Place in magnet cradle. Let stand 5 minutes, and pull down the magnetic bead.</p>
</li>
<li><p>Extract the assembled DNA. With a pipette, extract the supernatant containing the assembled DNA.</p>
</li>
<li><p>Add RDP Storage buffer. Add 20 μL of storage buffer to neutralize the sample for transformation and gel analysis.</p>
</li>
</ol>
<h3 id="follow-up-steps">Follow up steps</h3>
<p><strong>Gel analysis:</strong>
Gel analysis is a fast way of assessing the efficiency of the assembly and the yield of the product. For a good build the major gel band should also be the longest with a size corresponding to the predicted length of your design. Typically a 1% agarose gel will resolve fragments up to 10 kb. Even if the desired product is not the major band, don’t give up! The preponderance of smaller bands should be inactive and will therefore be under represented amongst in the total colony count.</p>
<p><strong>Cell Transformation:</strong>
In our hands, builds up to 8 parts routinely yield hundreds of colonies using chemically competent cells. Use the cell type and transformation procedure your lab feels comfortable with, but keep in mind that the more ambitious your build becomes, the fewer colonies you will get. In these situations you may want to consider using a high-efficiency transformation protocol.</p>
<p><strong>Miniprep Analysis:</strong>
The number of minipreps that are required to identify correct constructions will depend upon the number of parts used in the assembly (accuracy drops fractionally with the number of parts added). At Synbiota, we can generally achieve &gt;80% accuracy with an 8-part assembly (including Anchor + Cap). For assemblies up to 8 parts, plan on doing 4-6 minipreps to be on the safe side.</p>
<p>The final construct will contain BsaI sites within the Anchor and Cap parts that will precisely excise the circuit at X’ or Z’ ends that begin the circuit and the X or Z ends that complete the circuit. Cutting 100-200 ng of miniprep DNA will therefore result in a fragment that corresponds exactly to the size predicted by your design, and a fragment that corresponds to the plasmid backbone that will range in size from 2.2-2.5 kb, depending the Anchors and Caps chosen for the assembly. Include an uncut control lane on the gel to be sure the enzyme has cut efficiently and to avoid confusion in interpreting band identities.</p>
<p><strong>DNA Sequencing:</strong>
Parts produced by PCR from the high-fidelity thermostable polymerases Phusion and Pfu, introduce errors at a frequency that is less, but close to physiological (Phusion: ~5x10-7 Pfu: ~1x10-6), therefore circuit errors that are contained within parts that have been derived by PCR will be extremely rare. For parts that have been chemically synthesized, the error frequency is considerably higher (~1x10-3) but since the total amount of synthetic DNA used in any given assembly is small, the likely hood of achieving an error-free construct is high. Nonetheless prudent researchers will always determine the DNA sequence of a circuit design either before functional testing or after discovering that something about the design is not working.</p>
<p>Complete sequence coverage of parts or circuits that are &lt;1000 bp can typically be achieved using the BBa RFC-10 forward primer, BBa_G00100 (5’- tgccacctgacgtctaagaa) and its reverse counterpart, BBa_G00101 (5’- attaccgcctttgagtgagc).</p>
<p>The forward primer is located in all of the Cap parts provided, and is situated 184 bp upstream of the BsaI site. The reverse primer is only present in the high-copy Anchors, and is situated 168 bp downstream of the BsaI site*. For longer parts and circuits, the additional forward and reverse primers required for complete sequence coverage can be automatically generated using the RDP sequencing primer design software tool.</p>
<h3 id="references">References</h3>
<p>Protocol obtained from Synbiota: <a href="https://synbiota.com/beta">https://synbiota.com/beta</a></p>
<h2 id="ligase-cycling-reaction-protocol">Ligase Cycling Reaction Protocol</h2>
<p><em>Modified from a protocol written by Naveen Venayak</em></p>
<h3 id="safety-precautions">Safety Precautions</h3>
<p><strong>SDS (safety data sheet)</strong>: Refer to the SDS sheets for all listed materials before entering the lab. Be
prepared to answer any questions regarding the information on these sheets.</p>
<p><strong>PPE (Personal protective equipment)</strong>: Proper lab attire should be worn throughout the experiment: This
means that upon entering the lab you should be wearing long pants and close-toed shoes. Contact
lenses should not be worn. Furthermore, a lab coat, goggles, and gloves should be worn at all times,
and long hair should be tied back.</p>
<p><strong>Autoclave</strong>: The autoclave should only be handled by execs. <strong>Note that any autoclaved materials
may still be hot and should therefore be handled with caution. Be careful not to burn yourself.</strong></p>
<h3 id="reagents">Reagents</h3>
<ul>
<li>~15&#956;L DNA Parts</li>
<li>1.5&#956;M Bridging Olgios</li>
<li>3&#956;L Ampligase Buffer</li>
<li>1.5&#956;L Ampligase</li>
<li>2.25&#956;L Betaine</li>
<li>4&#956;L DMSO</li>
<li>26&#956;L Nuclease free water</li>
<li>1&#956;L ATP</li>
<li>1&#956;L T4 Polynucleotide Kinase</li>
</ul>
<h3 id="materials">Materials</h3>
<ul>
<li>Thermal Cycler</li>
</ul>
<h3 id="lab-protocol">Lab Protocol</h3>
<h4 id="part-1-primer-design">Part 1: Primer Design</h4>
<ol>
<li>Design primers to amplify fragments and if required, add any additional features</li>
<li>Bridging olgios designed with a 70<sup>o</sup>C melting temperature</li>
<li>Inserts were designed with UNS sequences (unique nucleotide sequences) for primer design. Sequences provided by Naveen Venayak.</li>
</ol>
<h4 id="part-2-dna-phosphorylation">Part 2: DNA Phosphorylation</h4>
<ol>
<li>Prepare 20&#956;L mix containing:<ul>
<li>100fmol of each purified DNA fragment</li>
<li>5mM ATP</li>
<li>10U T4 polynucleotide kinase in 1x Ampligase thermostable DNA ligase reactions buffer</li>
</ul>
</li>
<li>Incubate at 37<sup>o</sup>C for 1 hour to phosphorylate 5&#39; ends</li>
<li>Incubate 65<sup>o</sup>C for 20min to inactivate enzymes</li>
</ol>
<h4 id="part-3-ligase-cycling-reaction">Part 3: Ligase Cycling Reaction</h4>
<ol>
<li>Prepare master mix with:</li>
</ol>
<table>
<thead>
<tr>
<th style="text-align:center">Component</th>
<th style="text-align:center">Volume</th>
</tr>
</thead>
<tbody>
<tr>
<td style="text-align:center">DNA parts</td>
<td style="text-align:center">15&#956;L</td>
</tr>
<tr>
<td style="text-align:center">1.5&#956;M Bridging olgios</td>
<td style="text-align:center">0.5&#956;L each</td>
</tr>
<tr>
<td style="text-align:center">Ampligase buffer (10x)</td>
<td style="text-align:center">1&#956;L</td>
</tr>
<tr>
<td style="text-align:center">Ampligase (50U/&#956;L)</td>
<td style="text-align:center">1.5&#956;L</td>
</tr>
<tr>
<td style="text-align:center">5M Betaine</td>
<td style="text-align:center">2.25</td>
</tr>
<tr>
<td style="text-align:center">DMSO</td>
<td style="text-align:center">2&#956;L</td>
</tr>
<tr>
<td style="text-align:center">Water</td>
<td style="text-align:center">To 25&#956;L (4.75 - 0.5 * # of linker olgios)</td>
</tr>
</tbody>
</table>
<ol>
<li>Make master mix with:<ul>
<li>Ampligase Buffer (10&#956;l)</li>
<li>DMSO (20&#956;L)</li>
<li>5M Betaine (22.5&#956;L)</li>
<li>Total = 52.5 or 5.25&#956;L</li>
</ul>
</li>
<li>Set-up and running thermal cycler with these settings:<ul>
<li>94<sup>o</sup>C for 2 min</li>
<li>94<sup>o</sup>C for 10 sec</li>
<li>55<sup>o</sup>C for 30 sec</li>
<li>66<sup>o</sup>C for 60 sec</li>
<li>Go to 2, 49 times</li>
<li>Hold 4<sup>o</sup>C forever</li>
</ul>
</li>
</ol>
<h4 id="part-4-transformation">Part 4: Transformation</h4>
<ol>
<li>Chemically competent cells sufficed for &lt;8 pieces</li>
<li>Screen ~2 resulting colonies</li>
</ol>
<h3 id="leaving-the-lab">Leaving the Lab</h3>
<p>Prior to leaving the lab, you should:</p>
<ol>
<li>Clean dirty glassware, or at least set aside the glassware to be cleaned by a designated individual.</li>
<li>Wipe down your workspace.</li>
<li>Ensure that all materials have been returned to their places, and that the plates have been properly
stored in the fridge.</li>
</ol>
<h3 id="reference">Reference</h3>
<p>Protocol provided by Naveen Venayak</p>
</div></div><div id="tableofcontents" class="tableofcontents affix sidebar col-lg-4 hidden-xs hidden-sm hidden-md visible-lg-3"><ul class="nav">
<li><a href="#protocol-for-lb-medium-and-lb-agar-plates">Protocol for LB medium and LB agar plates</a><ul>
<li><a href="#lb-introduction">LB Introduction</a></li>
<li><a href="#lb-basic-terminology-and-concepts">LB Basic Terminology and Concepts</a></li>
<li><a href="#lb-safety-precautions">LB Safety Precautions</a></li>
<li><a href="#lb-materials-and-equipment">LB Materials and Equipment</a></li>
<li><a href="#lb-reagents">LB Reagents</a></li>
<li><a href="#lb-materials">LB Materials</a></li>
<li><a href="#lb-lab-protocol">LB Lab Protocol</a><ul>
<li><a href="#--part-1-making-the-lb-broth--"><strong>Part 1. Making the LB broth</strong></a></li>
<li><a href="#--part-2-autoclaving----only-done-by-execs-----"><strong>Part 2. Autoclaving *</strong>ONLY DONE BY EXECS<strong>*</strong></a></li>
<li><a href="#--part-3-pouring-the-plates--if-you-are-making-lb-agar---"><strong>Part 3. Pouring the plates (if you are making LB agar)</strong></a></li>
</ul>
</li>
<li><a href="#lb-leaving-the-lab">LB Leaving the lab</a></li>
<li><a href="#lb-references">LB References</a></li>
</ul>
</li>
<li><a href="#protocol-for-soc-medium">Protocol for SOC medium</a><ul>
<li><a href="#soc-introduction">SOC Introduction</a></li>
<li><a href="#soc-safety-precautions">SOC Safety Precautions</a></li>
<li><a href="#soc-materials-and-equipment">SOC Materials and Equipment</a></li>
<li><a href="#soc-lab-protocol">SOC Lab Protocol</a><ul>
<li><a href="#--part-1-making-the-soc-broth--"><strong>Part 1. Making the SOC broth</strong></a></li>
<li><a href="#---part-2--autoclaving-only-done-by-execs--"><strong> Part 2.  Autoclaving ONLY DONE BY EXECS</strong></a></li>
<li><a href="#--part-3--"><strong>Part 3</strong></a></li>
</ul>
</li>
<li><a href="#soc-leaving-the-lab">SOC Leaving the Lab</a></li>
<li><a href="#soc-references">SOC References</a></li>
</ul>
</li>
<li><a href="#king-s-b-protocol">King&#39;s B Protocol</a><ul>
<li><a href="#kb-introduction">KB Introduction</a></li>
<li><a href="#kb--basic-terminology-and-concepts">KB  Basic Terminology and Concepts</a></li>
<li><a href="#kb-safety-precautions">KB Safety Precautions</a></li>
<li><a href="#kb-materials-and-equipment">KB Materials and Equipment</a></li>
<li><a href="#kb-lab-protocol">KB Lab Protocol</a><ul>
<li><a href="#--part-1-making-king-s-b-broth--"><strong>Part 1. Making King&#39;s B broth</strong></a></li>
<li><a href="#--part-2-autoclaving----only-done-by-execs---"><strong>Part 2. Autoclaving *</strong>ONLY DONE BY EXECS<em>*</em></a></li>
<li><a href="#--part-3-pouring-the-plates--if-you-are-making-agar-plates---"><strong>Part 3. Pouring the plates (if you are making agar plates)</strong></a></li>
</ul>
</li>
<li><a href="#kb-leaving-the-lab">KB Leaving the Lab</a></li>
<li><a href="#kb-references">KB References</a></li>
</ul>
</li>
<li><a href="#antibiotic-stock-preparation">Antibiotic Stock Preparation</a><ul>
<li><a href="#--background--"><strong>Background</strong></a></li>
<li><a href="#--basic-terminology-and-concepts--"><strong>Basic Terminology and Concepts</strong></a></li>
<li><a href="#safety-precautions">Safety Precautions</a></li>
<li><a href="#materials-and-equipment">Materials and Equipment</a></li>
<li><a href="#procedure">Procedure</a><ul>
<li><a href="#ampicillin">Ampicillin</a></li>
<li><a href="#kanamycin">Kanamycin</a></li>
<li><a href="#tetracycline">Tetracycline</a></li>
<li><a href="#chloramphenicol">Chloramphenicol</a></li>
</ul>
</li>
<li><a href="#leaving-the-lab">Leaving the Lab</a></li>
<li><a href="#acknowledgement">Acknowledgement</a></li>
</ul>
</li>
<li><a href="#gel-electrophoresis">Gel Electrophoresis</a><ul>
<li><a href="#part-1--casting-the-gel">Part 1: Casting the gel</a></li>
<li><a href="#part-2--loading-the-gel">Part 2: Loading the gel</a></li>
<li><a href="#references-">References:</a></li>
</ul>
</li>
<li><a href="#making-chemically-competent-cells">Making Chemically Competent Cells</a><ul>
<li><a href="#materials">Materials</a></li>
<li><a href="#glassware---equipment">Glassware &amp; Equipment</a></li>
<li><a href="#protocol">Protocol</a><ul>
<li><a href="#--part-1--tss-buffer--"><strong>Part 1: TSS Buffer</strong></a></li>
<li><a href="#--part-2--making-chemically-competent-cells--"><strong>Part 2: Making Chemically Competent Cells</strong></a></li>
</ul>
</li>
</ul>
</li>
<li><a href="#lab-protocol-for-bacterial-transformation">Lab protocol for bacterial transformation</a><ul>
<li><a href="#introduction">Introduction</a></li>
<li><a href="#basic-terminology-and-concepts">Basic Terminology and concepts</a></li>
<li><a href="#safety-precautions">Safety precautions</a></li>
<li><a href="#reagents">Reagents</a></li>
<li><a href="#lab-protocol">Lab protocol</a></li>
<li><a href="#leaving-the-lab">Leaving the lab</a></li>
</ul>
</li>
<li><a href="#cell-lysis-protocol">Cell Lysis Protocol</a><ul>
<li><a href="#day-1">Day 1</a></li>
<li><a href="#day-2">Day 2</a></li>
<li><a href="#day-3---extraction-of-cell-lysates">Day 3 - Extraction of cell lysates</a></li>
</ul>
</li>
<li><a href="#sds-page-protocol">SDS-PAGE Protocol</a><ul>
<li><a href="#--introduction--"><strong>Introduction</strong></a></li>
<li><a href="#--basic-terminology-and-concepts--"><strong>Basic Terminology and Concepts</strong></a></li>
<li><a href="#--materials--"><strong>Materials</strong></a></li>
<li><a href="#equipment">Equipment</a></li>
<li><a href="#lab-protocol">Lab Protocol</a><ul>
<li><a href="#--preparation-of-1x-sds-page-running-buffer--"><strong>Preparation of 1X SDS-PAGE Running Buffer.</strong></a></li>
<li><a href="#--preparation-of-sds-page-gel--"><strong>Preparation of SDS-PAGE gel</strong></a></li>
<li><a href="#--running-the-sds-page-gel--"><strong>Running the SDS-PAGE gel</strong></a></li>
<li><a href="#--staining-the-sds-page-gel--"><strong>Staining the SDS-PAGE gel</strong></a></li>
</ul>
</li>
<li><a href="#--leaving-the-lab--"><strong>Leaving the lab</strong></a></li>
<li><a href="#references">References</a></li>
</ul>
</li>
<li><a href="#purelink--pcr-purification-protocol">PureLink® PCR Purification protocol</a><ul>
<li><a href="#reference">Reference</a></li>
</ul>
</li>
<li><a href="#quantum-prep-plasmid-miniprep-protocol">Quantum Prep Plasmid Miniprep Protocol</a><ul>
<li><a href="#reference">Reference</a></li>
</ul>
</li>
<li><a href="#pcr-based-rdp-parts-creation-protocol">PCR-Based RDP Parts Creation Protocol</a><ul>
<li><a href="#introduction">Introduction</a></li>
<li><a href="#buffers--reagents-and-consumables">Buffers, Reagents and Consumables</a></li>
<li><a href="#pcr-amplification">PCR Amplification</a></li>
<li><a href="#references">References</a></li>
</ul>
</li>
<li><a href="#bsai-digestion-protocol">BsaI Digestion Protocol</a><ul>
<li><a href="#introduction">Introduction</a></li>
<li><a href="#safety-precautions">Safety Precautions</a></li>
<li><a href="#reagents">Reagents</a></li>
<li><a href="#materials">Materials</a></li>
<li><a href="#lab-protocol">Lab Protocol</a><ul>
<li><a href="#part-1--bsai-digestion">Part 1: BsaI Digestion</a></li>
<li><a href="#part-2--dna-cleanup">Part 2: DNA Cleanup</a></li>
<li><a href="#part-3--measuring-and-adjusting-the-part-concentration">Part 3: Measuring and adjusting the part concentration</a></li>
<li><a href="#part-4--part-validation">Part 4: Part Validation</a></li>
</ul>
</li>
<li><a href="#leaving-the-lab">Leaving the Lab</a></li>
<li><a href="#references">References</a></li>
</ul>
</li>
<li><a href="#protocol-for-synbiota-rdp-assembly">Protocol for Synbiota RDP Assembly</a><ul>
<li><a href="#preparation">Preparation</a></li>
<li><a href="#rdp-assembly">RDP Assembly</a></li>
<li><a href="#follow-up-steps">Follow up steps</a></li>
<li><a href="#references">References</a></li>
</ul>
</li>
<li><a href="#ligase-cycling-reaction-protocol">Ligase Cycling Reaction Protocol</a><ul>
<li><a href="#safety-precautions">Safety Precautions</a></li>
<li><a href="#reagents">Reagents</a></li>
<li><a href="#materials">Materials</a></li>
<li><a href="#lab-protocol">Lab Protocol</a><ul>
<li><a href="#part-1--primer-design">Part 1: Primer Design</a></li>
<li><a href="#part-2--dna-phosphorylation">Part 2: DNA Phosphorylation</a></li>
<li><a href="#part-3--ligase-cycling-reaction">Part 3: Ligase Cycling Reaction</a></li>
<li><a href="#part-4--transformation">Part 4: Transformation</a></li>
</ul>
</li>
<li><a href="#leaving-the-lab">Leaving the Lab</a></li>
<li><a href="#reference">Reference</a></li>
</ul>
</li>
</ul>
</div></div></div>
</html>
{{Toronto/footer}}
