# ConsortiaFlux

Within nature, it is practically impossible to find a single bacterial species on its own. This is because bacteria exist in communities called a microbiome. However, in synthetic biology, a single strain is often grown independently in a laboratory setting for the intended use in a natural environment. Experimentally, this would require the creation of an isolated microbiome, or a simulation of one.

Our iGEM team created a web-based tool, 'ConsortiaFlux,' to run a microbiome simulation. The tool allows a user to visualize the effect of the addition of a synthetically engineered bacteria into a predefined microbiome. This visualization is generated through a process called community flux-balance analysis, or cFBA.

To demonstrate our web-based tool, we genetically engineered an E. coli bacteria for bioremediation. This engineered E. coli remediates toluene in Albertan tailing ponds, supplementing the existing microbiome, and providing a concrete example of how our tool can be applied.

Tying it all together is an analysis of the best technical, economic, and social applications of our technology within a solid framework.

## What is FBA and cFBA?

FBA, or flux-balance analysis, is a tool used to study genome-scale metabolic network reconstructions. By analyzing flow of metabolites within a given network, FBA computes the predicted production rate of a metabolite, or the growth rate. In a synthetic biology context, this becomes useful for assessing outcomes of a transformation in silico.

cFBA, or community based flux-balance analysis, is FBA at the community level. cFBA allows for the prediction of a genetic modification in a bacterial community.

## Math Behind the Code

FBA calcultes the flow of metabolites at steady state in which mass balance is not changing with time (i.e. (dx/dt)=0). In an FBA with m unique compounds and n reactions, a stoichiometric matrix(S) with size m*n represents all the reaction set. Vector fluxes to be computed are respresented by a vector v. An objective function (Z) is set to determine which flux to optimize in a given boundary. Hence following is computed linearly,

                                                Maximize     c^Tv
                                                Subject to   Sv=0
                                                Boundaries   lowerbound<v<upperbound

(Insert Image: FBA_diagram.jpg)
Taken from:
Title:What is flux balance analysis?
Author:Jeffrey D Orth, Ines Thiele, Bernhard Ø Palsson
Publication:Nature Biotechnology
Publisher:Nature Publishing Group
Date:Mar 1, 2010
Copyright © 2010, Rights Managed by Nature Publishing Group

Note: See software section for more


## Proof-of-Concept: A Genetically Engineered Solution for Oil Sand Tailings


### Background

As conventional oil production declines in the coming decades, unconventional
sources of oil such as bitumen will become increasingly important for supplying
oil. Currently, Canada possesses the largest of these bitumen reserves in
Alberta in the form of oil sands. Alberta’s oil sands have about 168 billion
barrels, making it the third largest crude oil reserve in the world. The use and
export of this oil significantly contributes to economic growth in Canada, and
will continue to do so.

### Economic impact of oil sands on Canada

* Total GDP impacts of all oil sands investment, re-investment and operating revenues is estimated to be $3,865 billion for Canada.
* Oil sands related direct employment in Alberta is expected to continue growing from the current level (2014) of 146,000 jobs to a peak of 256,000 jobs in 2024.
* For every direct job generated in the Alberta oil sands, 1 additional job is generated by indirect association and 1.5 jobs by induced association in Canada.

Despite this huge economic potential, the use of oil sands is raising concerns
regarding environmental costs, especially those associated with the creation of
oil sands tailings.   Since the oil sand tailings are essentially a dumping
site, it has a high concentration of toxic organic compounds. The Government of
Alberta has a zero discharge policy for open-mined oil operations. This means
that all oil sand process-affected water (OSPW) and tailings must be stored on
site, which leads to the accumulation of these toxic compounds . According to a
report released by WWF, this is a growing problem. From 2006 to 2009, weight
percentage of toxic benzene, toluene, ethylbenzene, xylene (BTEX) and Polycyclic
Aromatic Hydrocarbons (PAH) present in oil tailings have increased by 29.7% and
15.5% respectively.   Although, the oil sand tailing ponds is supposed to retain
the toxic water, a recent Federal study suggests that the waste might be
leaching into the ground water and contaminating the Athabasca River. It was
estimated that as much as 6.5 litres might be leaching from a single pond per
day. Current BTEX concentrations have already exceeded the guidelines provided
by Canadian Council for Ministers of Environment (CCME) and thus are toxic to
wildlife and humans in the long term.

BTEX can cause damage to:

* Liver
* Kidneys
* Eyes
* Central Nervous System

### Current Solutions

The current solutions for BTEX and PAH contamination includes:

* _Soil Vapor Extraction (SVE)_

 SVE is a physical process that involves sucking up vapors of contaminants. This
 reduces the vapor pressure of the contaminant hence shifting the equilibrium
 towards the vapor phase and causing any contaminant in the solid or liquid
 phase to evaporate. Since the effectiveness of SVE depends strongly on:   

 * The contaminant’s physical and chemical properties
 * Temperature in the subsurface
 * Soil properties

 So SVE is not suitable for all compounds and under all conditions.

* _Bioremediation using Microorganisms_

 This technique uses microbes in order to break down BTEX and PAH using enzymes
 produces by the microbes. This technique is advantageous because it is: cheap,
 occurs on site and produces non-toxic compounds such as carbon dioxide and
 acetate.  However the Bioremediation process is slow because of:

 * The high toxicity of the pollutant
 * The high concentration of the pollutant
 * Low solubility of the pollutant
 * Low availability of nutrients in the soil
 * Low concentration of dissolved Oxygen
 * Fewer electron acceptors.

 This essentially limits the bioremediation process.

### Our Project

Our project therefore aims to maximize the efficiency of the bioremediation
process by creating a synthetically engineered microorganism that can degrade
these toxic compounds. Our team focused specifically on toluene since it is one
of the major BTEXs present in these oil sand tailings and is one of the most
studied one’s.

### Toluene degradation in microbial communities

Many microorganisms that survive in these oil sand tailings use toluene as a nutrient source. They have therefore evolved several pathways that can degrade toluene. These can be broadly divided into two categories:

* Aerobic pathways
* Anaerobic pathways

We focused specifically on the pathway used by _Pseudomonas putida F1_ since it
is a well-studied biosafety level 1 organism and some studies suggest it is one
of the most efficient at toluene degradation.   _P. putida F1_ degrades toluene
through an aerobic pathway. In its pathway the reactions involving _todC1C2BA_,
_todD_, _todE_ and _todF_ are specific to _P. putida F1_ and are the limiting
steps of this pathway. Especially accumulation of 3-methylcatechol is known to have severe toxic effects for bacteria. Once 2-Hydroxy-2,4-pentadienoate is formed it can be
degraded by enzymes that can are present in most species including _E.coli_.   A
plasmid, pTDG602, for the first part of the pathway involving _todC1C2BA_ and
_todD_ has already been created by Zylstra, from whom we then obtained it. This
only leaves _todE_ and _todF_ for which no such biobrick exists that has been
well characterized and so we chose to focus on that part of the pathway.

## Experimental Design

### Our Objectives  

* To design and construct a plasmid that contains both _todE_ and _todF_.
* To transform _E.coli_ with our created plasmid.
* To confirm the synthesis of _todE_ and _todF_.
* To characterize the activity of _todE_ and _todF_ (if time permits).
* To co-transform _E.coli_ with both pTDG602 and our assembled plasmid (if time permits).

### Plasmid Design

<p class="image-wrapper">
{{image "lcrconstruct.png" directlink mode}}
</p>

The diagram above shows the plasmid that we designed. We used psB1C3, in green,
for the backbone that we obtained from the igem registry. The plasmid includes
both _todE_ and _todF_ that have been optimized for _E.coli_ K-12 MG1655 strain .
Each gene begins with a start codon followed by a His-Tag and terminates with a
stop codon. The purpose of the His-Tag is to confirm the synthesis of our
enzymes once we transform _E.coli_. We added a Ribosomal binding site (RBS)
before each gene and optimized it using the Salis RBS calculator. UNSs were also
added before and after every gene to prevent the formation of a secondary
structure.  

### Experimental details  

To create our plasmid we used Synbiota’s RDP protocol that allowed us to
assemble our _todE_ and _todF_ constructs. We confirmed the identity of our
constructs by using gel electrophoresis to compare our constructs against the
_todE_ and _todF_ genes we purified from _P. putida F1_. Once the _todE_ and
_todF_ genes have been assembled they are be inserted into the psB1C3 backbone
using Ligase Chain Reaction (LCR). Gel electrophoresis will once again be used
to ensure that our plasmid has been correctly assembled. Once our plasmid is
assembled we then transformed _E.coli_ that we will make chemically competent
for transformation. Antibiotic selection is  used to identify the
transformed colonies. To test for the synthesis of the enzymes, the transformed
colonies will be lysed and SDS-PAGE.   After
confirming the production of the enzymes, the transformed _E.coli_ will be grown
on a 3-methyl catechol rich media and then a colorimetric assay will be used to
identify the formation of 2-Hydroxy-6-keto-2,4-heptadienoate (substrate of
_todE_); and 2-Hydroxy-2,4-pentadienoate substrate of _todF_). Also, if time
permits Gas Chromatography and Mass spectrometry can also be used to
characterize the activity of these enzymes.   Furthermore, we aimed to co-transform
the bacteria with our designed plasmid and pTDG602 to observe a complete
degradation of toluene to carbon dioxide and water.
